[![Contributors][contributors-shield]][contributors-url]
[![Forks][forks-shield]][forks-url]
[![Stargazers][stars-shield]][stars-url]
[![Issues][issues-shield]][issues-url]

## Updated on 2024.03.20
> Usage instructions: [here](./docs/README.md#usage)

<details>
  <summary>Table of Contents</summary>
  <ol>
    <li><a href=#dti>DTI</a></li>
    <li><a href=#ddi>DDI</a></li>
    <li><a href=#mg>MG</a></li>
    <li><a href=#mpp>MPP</a></li>
    <li><a href=#md>MD</a></li>
    <li><a href=#transformer>Transformer</a></li>
    <li><a href=#bind>bind</a></li>
  </ol>
</details>

## DTI

|Publish Date|Title|Authors|PDF|Code|
|---|---|---|---|---|
|**2005-04-07**|**Design of HIV-1-PR inhibitors which do not create resistance: blocking the folding of single monomers**|R. A. Broglia et.al.|[q-bio/0504011](http://arxiv.org/abs/q-bio/0504011)|null|
|**2024-02-06**|**Tag-LLM: Repurposing General-Purpose LLMs for Specialized Domains**|Junhong Shen et.al.|[2402.05140](http://arxiv.org/abs/2402.05140)|**[link](https://github.com/sjunhongshen/tag-llm)**|
|**2024-01-30**|**A large dataset curation and benchmark for drug target interaction**|Alex Golts et.al.|[2401.17174](http://arxiv.org/abs/2401.17174)|null|
|**2024-03-13**|**Fast Dual-Regularized Autoencoder for Sparse Biological Data**|Aleksandar Poleksic et.al.|[2401.16664](http://arxiv.org/abs/2401.16664)|null|
|**2023-12-23**|**DTIAM: A unified framework for predicting drug-target interactions, binding affinities and activation/inhibition mechanisms**|Zhangli Lu et.al.|[2312.15252](http://arxiv.org/abs/2312.15252)|**[link](https://github.com/csubiogroup/dtiam)**|
|**2023-12-09**|**Learning to Denoise Unreliable Interactions for Link Prediction on Biomedical Knowledge Graph**|Tengfei Ma et.al.|[2312.06682](http://arxiv.org/abs/2312.06682)|null|
|**2024-03-19**|**Accurate and interpretable drug-drug interaction prediction enabled by knowledge subgraph learning**|Yaqing Wang et.al.|[2311.15056](http://arxiv.org/abs/2311.15056)|**[link](https://github.com/lars-research/knowddi)**|
|**2023-11-24**|**Towards a more inductive world for drug repurposing approaches**|Jesus de la Fuente et.al.|[2311.12670](http://arxiv.org/abs/2311.12670)|**[link](https://github.com/ubioinformat/graphemb)**|
|**2023-11-04**|**FragXsiteDTI: Revealing Responsible Segments in Drug-Target Interaction with Transformer-Driven Interpretation**|Ali Khodabandeh Yalabadi et.al.|[2311.02326](http://arxiv.org/abs/2311.02326)|**[link](https://github.com/yazdanimehdi/fragxsitedti)**|
|**2024-02-11**|**PGraphDTA: Improving Drug Target Interaction Prediction using Protein Language Models and Contact Maps**|Rakesh Bal et.al.|[2310.04017](http://arxiv.org/abs/2310.04017)|**[link](https://github.com/yijia-xiao/pgraphdta)**|
|**2023-09-28**|**Navigating Healthcare Insights: A Birds Eye View of Explainability with Knowledge Graphs**|Satvik Garg et.al.|[2309.16593](http://arxiv.org/abs/2309.16593)|null|
|**2023-07-11**|**Machine Learning Study of the Extended Drug-target Interaction Network informed by Pain Related Voltage-Gated Sodium Channels**|Long Chen et.al.|[2307.05794](http://arxiv.org/abs/2307.05794)|**[link](https://github.com/weilabmsu/oud-ppi)**|
|**2023-06-25**|**Meta-Path-based Probabilistic Soft Logic for Drug-Target Interaction Prediction**|Shengming Zhang et.al.|[2306.13770](http://arxiv.org/abs/2306.13770)|null|
|**2023-11-14**|**Causal Intervention for Measuring Confidence in Drug-Target Interaction Prediction**|Wenting Ye et.al.|[2306.00041](http://arxiv.org/abs/2306.00041)|null|
|**2023-10-18**|**A Federated Learning Benchmark for Drug-Target Interaction**|Gianluca Mittone et.al.|[2302.07684](http://arxiv.org/abs/2302.07684)|**[link](https://github.com/giemp95/feddti)**|
|**2023-03-21**|**Fine-Grained Selective Similarity Integration for Drug-Target Interaction Prediction**|Bin Liu et.al.|[2212.00543](http://arxiv.org/abs/2212.00543)|**[link](https://github.com/Nanfeizhilu/FGS_DTI_Predition)**|
|**2023-11-06**|**BatmanNet: Bi-branch Masked Graph Transformer Autoencoder for Molecular Representation**|Zhen Wang et.al.|[2211.13979](http://arxiv.org/abs/2211.13979)|**[link](https://github.com/wz-create/batmannet)**|
|**2022-10-20**|**A Methodology for the Prediction of Drug Target Interaction using CDK Descriptors**|Tanya Liyaqat et.al.|[2210.11482](http://arxiv.org/abs/2210.11482)|null|
|**2022-10-18**|**ImDrug: A Benchmark for Deep Imbalanced Learning in AI-aided Drug Discovery**|Lanqing Li et.al.|[2209.07921](http://arxiv.org/abs/2209.07921)|**[link](https://github.com/DrugLT/ImDrug)**|
|**2022-08-14**|**Widely Used and Fast De Novo Drug Design by a Protein Sequence-Based Reinforcement Learning Model**|Yaqin Li et.al.|[2209.07405](http://arxiv.org/abs/2209.07405)|null|
|**2023-01-19**|**Interpretable bilinear attention network with domain adaptation improves drug-target prediction**|Peizhen Bai et.al.|[2208.02194](http://arxiv.org/abs/2208.02194)|**[link](https://github.com/peizhenbai/drugban)**|
|**2023-12-15**|**Associative Learning Mechanism for Drug-Target Interaction Prediction**|Zhiqin Zhu et.al.|[2205.15364](http://arxiv.org/abs/2205.15364)|null|
|**2022-08-22**|**CandidateDrug4Cancer: An Open Molecular Graph Learning Benchmark on Drug Discovery for Cancer**|Xianbin Ye et.al.|[2203.00836](http://arxiv.org/abs/2203.00836)|null|
|**2022-02-14**|**Novel prediction methods for virtual drug screening**|Josip Mesarić et.al.|[2202.06635](http://arxiv.org/abs/2202.06635)|null|
|**2022-10-21**|**Target-aware Molecular Graph Generation**|Cheng Tan et.al.|[2202.04829](http://arxiv.org/abs/2202.04829)|null|
|**2022-01-16**|**Mitigating cold start problems in drug-target affinity prediction with interaction knowledge transferring**|Tri Minh Nguyen et.al.|[2202.01195](http://arxiv.org/abs/2202.01195)|**[link](https://github.com/ngminhtri0394/c2p2)**|
|**2022-08-09**|**Multiple Similarity Drug-Target Interaction Prediction with Random Walks and Matrix Factorization**|Bin Liu et.al.|[2201.09508](http://arxiv.org/abs/2201.09508)|**[link](https://github.com/intelligence-csd-auth-gr/DTI_MDMF2A)**|
|**2022-11-09**|**AI-Bind: Improving Binding Predictions for Novel Protein Targets and Ligands**|Ayan Chatterjee et.al.|[2112.13168](http://arxiv.org/abs/2112.13168)|**[link](https://github.com/Barabasi-Lab/AI-Bind)**|
|**2021-12-16**|**HampDTI: a heterogeneous graph automatic meta-path learning method for drug-target interaction prediction**|Hongzhun Wang et.al.|[2112.08567](http://arxiv.org/abs/2112.08567)|null|
|**2021-10-31**|**R-BERT-CNN: Drug-target interactions extraction from biomedical literature**|Jehad Aldahdooh et.al.|[2111.00611](http://arxiv.org/abs/2111.00611)|null|
|**2021-10-15**|**Improved Drug-target Interaction Prediction with Intermolecular Graph Transformer**|Siyuan Liu et.al.|[2110.07347](http://arxiv.org/abs/2110.07347)|null|
|**2021-09-27**|**Discovering Drug-Target Interaction Knowledge from Biomedical Literature**|Yutai Hou et.al.|[2109.13187](http://arxiv.org/abs/2109.13187)|null|
|**2021-09-17**|**An Interpretable Framework for Drug-Target Interaction with Gated Cross Attention**|Yeachan Kim et.al.|[2109.08360](http://arxiv.org/abs/2109.08360)|null|
|**2021-09-08**|**Machine learning modeling of family wide enzyme-substrate specificity screens**|Samuel Goldman et.al.|[2109.03900](http://arxiv.org/abs/2109.03900)|**[link](https://github.com/samgoldman97/enz-pred)**|
|**2021-07-10**|**Drug-Target Interaction Prediction with Graph Attention networks**|Haiyang Wang et.al.|[2107.06099](http://arxiv.org/abs/2107.06099)|**[link](https://github.com/Haiyang-W/DTI-GRAPH)**|
|**2021-11-19**|**Toward Drug-Target Interaction Prediction via Ensemble Modeling and Transfer Learning**|Po-Yu Kao et.al.|[2107.00719](http://arxiv.org/abs/2107.00719)|null|
|**2021-06-27**|**DGL-LifeSci: An Open-Source Toolkit for Deep Learning on Graphs in Life Science**|Mufei Li et.al.|[2106.14232](http://arxiv.org/abs/2106.14232)|**[link](https://github.com/awslabs/dgl-lifesci)**|
|**2022-01-15**|**Optimizing Area Under the Curve Measures via Matrix Factorization for Predicting Drug-Target Interaction with Multiple Similarities**|Bin Liu et.al.|[2105.01545](http://arxiv.org/abs/2105.01545)|**[link](https://github.com/intelligence-csd-auth-gr/DTI_MF2A)**|
|**2022-02-24**|**Heterogeneous Graph based Deep Learning for Biomedical Network Link Prediction**|Jinjiang Guo et.al.|[2102.01649](http://arxiv.org/abs/2102.01649)|null|
|**2021-07-09**|**Drug-Target Interaction Prediction via an Ensemble of Weighted Nearest Neighbors with Interaction Recovery**|Bin Liu et.al.|[2012.12325](http://arxiv.org/abs/2012.12325)|**[link](https://github.com/Nanfeizhilu/DTI_WkNNIR)**|
|**2020-12-21**|**Learn molecular representations from large-scale unlabeled molecules for drug discovery**|Pengyong Li et.al.|[2012.11175](http://arxiv.org/abs/2012.11175)|null|
|**2020-11-28**|**Discovering Synergistic Drug Combinations for COVID with Biological Bottleneck Models**|Wengong Jin et.al.|[2011.04651](http://arxiv.org/abs/2011.04651)|null|
|**2023-11-16**|**Deep learning analysis of intracranial EEG for recognizing drug effects and mechanisms of action**|Konstantin Y. Kalitin et.al.|[2009.12984](http://arxiv.org/abs/2009.12984)|null|
|**2020-09-28**|**GEFA: Early Fusion Approach in Drug-Target Affinity Prediction**|Tri Minh Nguyen et.al.|[2009.12146](http://arxiv.org/abs/2009.12146)|**[link](https://github.com/ngminhtri0394/GEFA)**|
|**2021-02-20**|**DTI-SNNFRA: Drug-Target interaction prediction by shared nearest neighbors and fuzzy-rough approximation**|Sk Mazharul Islam et.al.|[2009.10766](http://arxiv.org/abs/2009.10766)|null|
|**2021-12-13**|**PIGNet: A physics-informed deep learning model toward generalized drug-target interaction predictions**|Seokhyun Moon et.al.|[2008.12249](http://arxiv.org/abs/2008.12249)|**[link](https://github.com/jaechanglim/DTI_PDBbind)**|
|**2020-07-31**|**A computational approach to aid clinicians in selecting anti-viral drugs for COVID-19 trials**|Aanchal Mongia et.al.|[2007.01902](http://arxiv.org/abs/2007.01902)|**[link](https://github.com/aanchalMongia/DVA)**|
|**2020-08-23**|**Multi-View Self-Attention for Interpretable Drug-Target Interaction Prediction**|Brighter Agyemang et.al.|[2005.00397](http://arxiv.org/abs/2005.00397)|**[link](https://github.com/bbrighttaer/jova_baselines)**|
|**2020-04-26**|**Exploring the SARS-CoV-2 virus-host-drug interactome for drug repurposing**|Sepideh Sadegh et.al.|[2004.12420](http://arxiv.org/abs/2004.12420)|null|
|**2020-04-23**|**MolTrans: Molecular Interaction Transformer for Drug Target Interaction Prediction**|Kexin Huang et.al.|[2004.11424](http://arxiv.org/abs/2004.11424)|**[link](https://github.com/kexinhuang12345/MolTrans)**|
|**2020-12-09**|**DeepPurpose: a Deep Learning Library for Drug-Target Interaction Prediction**|Kexin Huang et.al.|[2004.08919](http://arxiv.org/abs/2004.08919)|**[link](https://github.com/kexinhuang12345/DeepPurpose)**|
|**2019-12-06**|**Drug-Target Indication Prediction by Integrating End-to-End Learning and Fingerprints**|Brighter Agyemang et.al.|[1912.01163](http://arxiv.org/abs/1912.01163)|**[link](https://github.com/bbrighttaer/ivpgan)**|
|**2021-06-07**|**Spectral Geometric Matrix Completion**|Amit Boyarski et.al.|[1911.07255](http://arxiv.org/abs/1911.07255)|null|
|**2019-08-15**|**Self-Attention Based Molecule Representation for Predicting Drug-Target Interaction**|Bonggun Shin et.al.|[1908.06760](http://arxiv.org/abs/1908.06760)|null|
|**2019-06-22**|**BERE: An accurate distantly supervised biomedical entity relation extraction network**|Lixiang Hong et.al.|[1906.06916](http://arxiv.org/abs/1906.06916)|null|
|**2019-04-17**|**Predicting drug-target interaction using 3D structure-embedded graph representations from graph neural networks**|Jaechang Lim et.al.|[1904.08144](http://arxiv.org/abs/1904.08144)|null|
|**2018-11-06**|**DeepConv-DTI: Prediction of drug-target interactions via deep learning with convolution on protein sequences**|Ingoo Lee et.al.|[1811.02114](http://arxiv.org/abs/1811.02114)|null|
|**2020-12-18**|**ChemBoost: A chemical language based approach for protein-ligand binding affinity prediction**|Rıza Özçelik et.al.|[1811.00761](http://arxiv.org/abs/1811.00761)|**[link](https://github.com/boun-tabi/chemboost)**|
|**2019-08-21**|**PADME: A Deep Learning-based Framework for Drug-Target Interaction Prediction**|Qingyuan Feng et.al.|[1807.09741](http://arxiv.org/abs/1807.09741)|**[link](https://github.com/simonfqy/PADME)**|
|**2018-12-08**|**DeepAffinity: Interpretable Deep Learning of Compound-Protein Affinity through Unified Recurrent and Convolutional Neural Networks**|Mostafa Karimi et.al.|[1806.07537](http://arxiv.org/abs/1806.07537)|**[link](https://github.com/Shen-Lab/DeepAffinity)**|
|**2018-08-31**|**FRnet-DTI: Deep Convolutional Neural Networks with Evolutionary and Structural Features for Drug-Target Interaction**|Farshid Rayhan et.al.|[1806.07174](http://arxiv.org/abs/1806.07174)|null|
|**2017-11-01**|**Erratum: Link prediction in drug-target interactions network using similarity indices**|Yiding Lu et.al.|[1711.00150](http://arxiv.org/abs/1711.00150)|null|
|**2017-07-04**|**iDTI-ESBoost: Identification of Drug Target Interaction Using Evolutionary and Structural Features with Boosting**|Farshid Rayhan et.al.|[1707.00994](http://arxiv.org/abs/1707.00994)|null|
|**2018-05-03**|**A generalized method toward drug-target interaction prediction via low-rank matrix projection**|Ratha Pech et.al.|[1706.01876](http://arxiv.org/abs/1706.01876)|**[link](https://github.com/rathapech/DTI_LMP)**|
|**2017-04-19**|**Fast Kronecker product kernel methods via generalized vec trick**|Antti Airola et.al.|[1601.01507](http://arxiv.org/abs/1601.01507)|null|
|**2015-03-12**|**Classification and its applications for drug-target interaction identification**|Jian-Ping Mei et.al.|[1502.04469](http://arxiv.org/abs/1502.04469)|null|

<p align=right>(<a href=#updated-on-20240320>back to top</a>)</p>

## DDI

|Publish Date|Title|Authors|PDF|Code|
|---|---|---|---|---|
|**2024-03-13**|**Learning to Describe for Predicting Zero-shot Drug-Drug Interactions**|Fangqi Zhu et.al.|[2403.08377](http://arxiv.org/abs/2403.08377)|**[link](https://github.com/zhufq00/ddis-prediction)**|
|**2024-02-28**|**Hierarchical Multi-Relational Graph Representation Learning for Large-Scale Prediction of Drug-Drug Interactions**|Mengying Jiang et.al.|[2402.18127](http://arxiv.org/abs/2402.18127)|**[link](https://github.com/mengyingjiang/hmgrl)**|
|**2024-02-21**|**Contextual Molecule Representation Learning from Chemical Reaction Knowledge**|Han Tang et.al.|[2402.13779](http://arxiv.org/abs/2402.13779)|null|
|**2024-02-18**|**DDIPrompt: Drug-Drug Interaction Event Prediction based on Graph Prompt Learning**|Yingying Wang et.al.|[2402.11472](http://arxiv.org/abs/2402.11472)|null|
|**2024-02-05**|**Comparative Analysis of LLaMA and ChatGPT Embeddings for Molecule Embedding**|Shaghayegh Sadeghi et.al.|[2402.00024](http://arxiv.org/abs/2402.00024)|**[link](https://github.com/sshaghayeghs/llama-vs-chatgpt)**|
|**2023-12-09**|**ALGNet: Attention Light Graph Memory Network for Medical Recommendation System**|Minh-Van Nguyen et.al.|[2312.08377](http://arxiv.org/abs/2312.08377)|**[link](https://github.com/tivivui95/algnet)**|
|**2023-12-09**|**Learning to Denoise Unreliable Interactions for Link Prediction on Biomedical Knowledge Graph**|Tengfei Ma et.al.|[2312.06682](http://arxiv.org/abs/2312.06682)|null|
|**2024-03-19**|**Accurate and interpretable drug-drug interaction prediction enabled by knowledge subgraph learning**|Yaqing Wang et.al.|[2311.15056](http://arxiv.org/abs/2311.15056)|**[link](https://github.com/lars-research/knowddi)**|
|**2023-11-15**|**Emerging Drug Interaction Prediction Enabled by Flow-based Graph Neural Network with Biomedical Network**|Yongqi Zhang et.al.|[2311.09261](http://arxiv.org/abs/2311.09261)|**[link](https://github.com/lars-research/emergnn)**|
|**2023-10-22**|**UniMAP: Universal SMILES-Graph Representation Learning**|Shikun Feng et.al.|[2310.14216](http://arxiv.org/abs/2310.14216)|**[link](https://github.com/fengshikun/unimap)**|
|**2023-09-28**|**Navigating Healthcare Insights: A Birds Eye View of Explainability with Knowledge Graphs**|Satvik Garg et.al.|[2309.16593](http://arxiv.org/abs/2309.16593)|null|
|**2023-08-26**|**Drug Interaction Vectors Neural Network: DrIVeNN**|Natalie Wang et.al.|[2308.13891](http://arxiv.org/abs/2308.13891)|null|
|**2023-08-11**|**Predicting Drug-Drug Interactions Using Knowledge Graphs**|Lizzy Farrugia et.al.|[2308.04172](http://arxiv.org/abs/2308.04172)|null|
|**2023-07-28**|**Co-attention Graph Pooling for Efficient Pairwise Graph Interaction Learning**|Junhyun Lee et.al.|[2307.15377](http://arxiv.org/abs/2307.15377)|**[link](https://github.com/leejunhyun/coattentiongraphpooling)**|
|**2023-08-18**|**Relation-aware graph structure embedding with co-contrastive learning for drug-drug interaction prediction**|Mengying Jiang et.al.|[2307.01507](http://arxiv.org/abs/2307.01507)|null|
|**2023-06-08**|**Comprehensive evaluation of deep and graph learning on drug-drug interactions prediction**|Xuan Lin et.al.|[2306.05257](http://arxiv.org/abs/2306.05257)|**[link](https://github.com/xzenglab/resources-for-ddis-prediction-using-dl)**|
|**2023-03-04**|**Decision Support System for Chronic Diseases Based on Drug-Drug Interactions**|Tian Bian et.al.|[2303.02405](http://arxiv.org/abs/2303.02405)|**[link](https://github.com/tianbian95/dssddi)**|
|**2023-02-22**|**Principled Drug-Drug Interaction Terms for Bayesian Logistic Regression Models of Drug Safety in Oncology Phase I Combination Trials**|Lukas A. Widmer et.al.|[2302.11437](http://arxiv.org/abs/2302.11437)|**[link](https://github.com/luwidmer/ddi-manuscript)**|
|**2023-02-17**|**Modeling Polypharmacy and Predicting Drug-Drug Interactions using Deep Generative Models on Multimodal Graphs**|Nhat Khang Ngo et.al.|[2302.08680](http://arxiv.org/abs/2302.08680)|**[link](https://github.com/hysonlab/drug-interactions)**|
|**2023-02-07**|**AMFPMC -- An improved method of detecting multiple types of drug-drug interactions using only known drug-drug interactions**|Bar Vered et.al.|[2302.03355](http://arxiv.org/abs/2302.03355)|null|
|**2023-02-06**|**Predicting Molecule-Target Interaction by Learning Biomedical Network and Molecule Representations**|Jinjiang Guo et.al.|[2302.00981](http://arxiv.org/abs/2302.00981)|null|
|**2022-12-21**|**Integrating Heterogeneous Domain Information into Relation Extraction: A Case Study on Drug-Drug Interaction Extraction**|Masaki Asada et.al.|[2212.10714](http://arxiv.org/abs/2212.10714)|null|
|**2024-02-23**|**Medical Knowledge Graph QA for Drug-Drug Interaction Prediction based on Multi-hop Machine Reading Comprehension**|Peng Gao et.al.|[2212.09400](http://arxiv.org/abs/2212.09400)|null|
|**2023-11-06**|**BatmanNet: Bi-branch Masked Graph Transformer Autoencoder for Molecular Representation**|Zhen Wang et.al.|[2211.13979](http://arxiv.org/abs/2211.13979)|**[link](https://github.com/wz-create/batmannet)**|
|**2022-10-19**|**Graph Regularized Probabilistic Matrix Factorization for Drug-Drug Interactions Prediction**|Stuti Jain et.al.|[2210.10784](http://arxiv.org/abs/2210.10784)|null|
|**2022-10-30**|**Predicting Drug-Drug Interactions using Deep Generative Models on Graphs**|Nhat Khang Ngo et.al.|[2209.09941](http://arxiv.org/abs/2209.09941)|**[link](https://github.com/hysonlab/drug-interactions)**|
|**2022-11-24**|**Distributed representations of graphs for drug pair scoring**|Paul Scherer et.al.|[2209.09383](http://arxiv.org/abs/2209.09383)|**[link](https://github.com/paulmorio/drugpairscoringdr)**|
|**2022-08-30**|**Graph Distance Neural Networks for Predicting Multiple Drug Interactions**|Haifan zhou et.al.|[2208.14810](http://arxiv.org/abs/2208.14810)|null|
|**2022-08-25**|**Molecular Substructure-Aware Network for Drug-Drug Interaction Prediction**|Xinyu Zhu et.al.|[2208.11267](http://arxiv.org/abs/2208.11267)|**[link](https://github.com/Hienyriux/MSAN)**|
|**2022-07-19**|**Drug prescription clusters in the UK Biobank: An assessment of drug-drug interactions and patient outcomes in a large patient cohort**|Kyriakos Schwarz et.al.|[2207.08665](http://arxiv.org/abs/2207.08665)|null|
|**2022-07-12**|**DDI Prediction via Heterogeneous Graph Attention Networks**|Farhan Tanvir et.al.|[2207.05672](http://arxiv.org/abs/2207.05672)|null|
|**2023-04-18**|**HyGNN: Drug-Drug Interaction Prediction via Hypergraph Neural Network**|Khaled Mohammed Saifuddin et.al.|[2206.12747](http://arxiv.org/abs/2206.12747)|null|
|**2022-10-07**|**Knowledge4COVID-19: A Semantic-based Approach for Constructing a COVID-19 related Knowledge Graph from Various Sources and Analysing Treatments' Toxicities**|Ahmad Sakor et.al.|[2206.07375](http://arxiv.org/abs/2206.07375)|**[link](https://github.com/sdm-tib/knowledge4covid-19)**|
|**2024-01-16**|**Integrating complex selection rules into the latent overlapping group Lasso for constructing coherent prediction models**|Guanbo Wang et.al.|[2206.05337](http://arxiv.org/abs/2206.05337)|**[link](https://github.com/guanbo-w/selectiondictionary)**|
|**2023-02-22**|**Graph neural networks and attention-based CNN-LSTM for protein classification**|Zhuangwei Shi et.al.|[2204.09486](http://arxiv.org/abs/2204.09486)|**[link](https://github.com/zshicode/gnn-attcl-protein)**|
|**2022-03-28**|**Multi-View Substructure Learning for Drug-Drug Interaction Prediction**|Zimeng Li et.al.|[2203.14513](http://arxiv.org/abs/2203.14513)|null|
|**2022-02-01**|**AI-based Approach for Safety Signals Detection from Social Networks: Application to the Levothyrox Scandal in 2017 on Doctissimo Forum**|Valentin Roche et.al.|[2203.03538](http://arxiv.org/abs/2203.03538)|null|
|**2022-09-19**|**Robust Causal Inference of Drug-drug Interactions**|Di Shu et.al.|[2203.00062](http://arxiv.org/abs/2203.00062)|null|
|**2022-05-26**|**ChemicalX: A Deep Learning Library for Drug Pair Scoring**|Benedek Rozemberczki et.al.|[2202.05240](http://arxiv.org/abs/2202.05240)|**[link](https://github.com/AstraZeneca/chemicalx)**|
|**2023-04-04**|**Central-Smoothing Hypergraph Neural Networks for Predicting Drug-Drug Interactions**|Duc Anh Nguyen et.al.|[2112.07837](http://arxiv.org/abs/2112.07837)|null|
|**2021-12-05**|**STNN-DDI: A Substructure-aware Tensor Neural Network to Predict Drug-Drug Interactions**|Hui Yu et.al.|[2111.05708](http://arxiv.org/abs/2111.05708)|**[link](https://github.com/zsy-9/STNN-DDI)**|
|**2021-12-11**|**A Unified View of Relational Deep Learning for Drug Pair Scoring**|Benedek Rozemberczki et.al.|[2111.02916](http://arxiv.org/abs/2111.02916)|**[link](https://github.com/astrazeneca/polypharmacy-ddi-synergy-survey)**|
|**2022-04-18**|**Multilingual Molecular Representation Learning via Contrastive Pre-training**|Zhihui Guo et.al.|[2109.08830](http://arxiv.org/abs/2109.08830)|null|
|**2023-07-21**|**ClueReader: Heterogeneous Graph Attention Network for Multi-hop Machine Reading Comprehension**|Peng Gao et.al.|[2107.00841](http://arxiv.org/abs/2107.00841)|null|
|**2021-06-01**|**An introduction to network analysis for studies of medication use**|Mohsen Askar et.al.|[2106.00413](http://arxiv.org/abs/2106.00413)|null|
|**2021-05-28**|**Relation Matters in Sampling: A Scalable Multi-Relational Graph Neural Network for Drug-Drug Interaction Prediction**|Arthur Feeney et.al.|[2105.13975](http://arxiv.org/abs/2105.13975)|null|
|**2022-07-17**|**SafeDrug: Dual Molecular Graph Encoders for Recommending Effective and Safe Drug Combinations**|Chaoqi Yang et.al.|[2105.02711](http://arxiv.org/abs/2105.02711)|null|
|**2021-03-19**|**Predicting Drug-Drug Interactions from Heterogeneous Data: An Embedding Approach**|Devendra Singh Dhami et.al.|[2103.10916](http://arxiv.org/abs/2103.10916)|null|
|**2021-01-25**|**EGFI: Drug-Drug Interaction Extraction and Generation with Fusion of Enriched Entity and Sentence Information**|Lei Huang et.al.|[2101.09914](http://arxiv.org/abs/2101.09914)|null|
|**2020-12-24**|**AttentionDDI: Siamese Attention-based Deep Learning method for drug-drug interaction predictions**|Kyriakos Schwarz et.al.|[2012.13248](http://arxiv.org/abs/2012.13248)|**[link](https://github.com/uzh-dqbm-cmi/side-effects)**|
|**2020-12-21**|**BERTChem-DDI : Improved Drug-Drug Interaction Prediction from text using Chemical Structure Information**|Ishani Mondal et.al.|[2012.11599](http://arxiv.org/abs/2012.11599)|null|
|**2020-12-21**|**Learn molecular representations from large-scale unlabeled molecules for drug discovery**|Pengyong Li et.al.|[2012.11175](http://arxiv.org/abs/2012.11175)|null|
|**2020-12-21**|**Towards Incorporating Entity-specific Knowledge Graph Information in Predicting Drug-Drug Interactions**|Ishani Mondal et.al.|[2012.11142](http://arxiv.org/abs/2012.11142)|null|
|**2021-04-10**|**Multi-view Graph Contrastive Representation Learning for Drug-Drug Interaction Prediction**|Yingheng Wang et.al.|[2010.11711](http://arxiv.org/abs/2010.11711)|**[link](https://github.com/isjakewong/MIRACLE)**|
|**2021-05-06**|**SumGNN: Multi-typed Drug Interaction Prediction via Efficient Knowledge Graph Summarization**|Yue Yu et.al.|[2010.01450](http://arxiv.org/abs/2010.01450)|**[link](https://github.com/yueyu1030/SumGNN)**|
|**2020-09-20**|**Relation Extraction from Biomedical and Clinical Text: Unified Multitask Learning Framework**|Shweta Yadav et.al.|[2009.09509](http://arxiv.org/abs/2009.09509)|null|
|**2020-08-31**|**Data Mining and Analytical Models to Predict and Identify Adverse Drug-drug Interactions**|Ricky Wang et.al.|[2009.00107](http://arxiv.org/abs/2009.00107)|null|
|**2020-08-28**|**PREMIER: Personalized REcommendation for Medical prescrIptions from Electronic Records**|Suman Bhoi et.al.|[2008.13569](http://arxiv.org/abs/2008.13569)|null|
|**2020-08-28**|**Two Step Joint Model for Drug Drug Interaction Extraction**|Siliang Tang et.al.|[2008.12704](http://arxiv.org/abs/2008.12704)|null|
|**2020-07-31**|**Screening and evaluation of potential clinically significant HIV drug combinations against SARS-CoV-2 virus**|Draško Tomić et.al.|[2007.16177](http://arxiv.org/abs/2007.16177)|null|
|**2020-07-05**|**DrugDBEmbed : Semantic Queries on Relational Database using Supervised Column Encodings**|Bortik Bandyopadhyay et.al.|[2007.02384](http://arxiv.org/abs/2007.02384)|null|
|**2020-07-04**|**Discovering Drug-Drug and Drug-Disease Interactions Inducing Acute Kidney Injury Using Deep Rule Forests**|Bowen Kuo et.al.|[2007.02103](http://arxiv.org/abs/2007.02103)|null|
|**2020-06-11**|**Bi-Level Graph Neural Networks for Drug-Drug Interaction Prediction**|Yunsheng Bai et.al.|[2006.14002](http://arxiv.org/abs/2006.14002)|null|
|**2020-10-29**|**Learning to Extrapolate Knowledge: Transductive Few-shot Out-of-Graph Link Prediction**|Jinheon Baek et.al.|[2006.06648](http://arxiv.org/abs/2006.06648)|**[link](https://github.com/JinheonBaek/GEN)**|
|**2020-05-20**|**A Minimal-Input Multilayer Perceptron for Predicting Drug-Drug Interactions Without Knowledge of Drug Structure**|Alun Stokes et.al.|[2005.10644](http://arxiv.org/abs/2005.10644)|null|
|**2020-05-12**|**GoGNN: Graph of Graphs Neural Network for Predicting Structured Entity Interactions**|Hanchen Wang et.al.|[2005.05537](http://arxiv.org/abs/2005.05537)|**[link](https://github.com/Hanchen-Wang/GoGNN)**|
|**2020-10-16**|**Drug-Drug Interaction Prediction with Wasserstein Adversarial Autoencoder-based Knowledge Graph Embeddings**|Yuanfei Dai et.al.|[2004.07341](http://arxiv.org/abs/2004.07341)|null|
|**2020-02-25**|**A Node Embedding Framework for Integration of Similarity-based Drug Combination Prediction**|Liang Yu et.al.|[2002.10625](http://arxiv.org/abs/2002.10625)|null|
|**2020-04-21**|**BiOnt: Deep Learning using Multiple Biomedical Ontologies for Relation Extraction**|Diana Sousa et.al.|[2001.07139](http://arxiv.org/abs/2001.07139)|**[link](https://github.com/lasigeBioTM/BiONT)**|
|**2019-12-08**|**Graph-augmented Convolutional Networks on Drug-Drug Interactions Prediction**|Yi Zhong et.al.|[1912.03702](http://arxiv.org/abs/1912.03702)|null|
|**2019-12-06**|**An Informatics-based Approach to Identify Key Pharmacological Components in Drug-Drug Interactions**|Jianyuan Deng et.al.|[1912.02964](http://arxiv.org/abs/1912.02964)|null|
|**2019-11-20**|**CASTER: Predicting Drug Interactions with Chemical Substructure Representation**|Kexin Huang et.al.|[1911.06446](http://arxiv.org/abs/1911.06446)|**[link](https://github.com/kexinhuang12345/CASTER)**|
|**2020-06-29**|**Beyond Textual Data: Predicting Drug-Drug Interactions from Molecular Structure Images using Siamese Neural Networks**|Devendra Singh Dhami et.al.|[1911.06356](http://arxiv.org/abs/1911.06356)|null|
|**2019-11-04**|**Attention-Gated Graph Convolutions for Extracting Drug Interaction Information from Drug Labels**|Tung Tran et.al.|[1910.12419](http://arxiv.org/abs/1910.12419)|null|
|**2019-10-08**|**GENN: Predicting Correlated Drug-drug Interactions with Graph Energy Neural Networks**|Tengfei Ma et.al.|[1910.02107](http://arxiv.org/abs/1910.02107)|null|
|**2020-07-07**|**SUPP.AI: Finding Evidence for Supplement-Drug Interactions**|Lucy Lu Wang et.al.|[1909.08135](http://arxiv.org/abs/1909.08135)|**[link](https://github.com/allenai/sdi-detection)**|
|**2019-08-04**|**Drug-Drug Interaction Prediction Based on Knowledge Graph Embeddings and Convolutional-LSTM Network**|Md. Rezaul Karim et.al.|[1908.01288](http://arxiv.org/abs/1908.01288)|**[link](https://github.com/rezacsedu/DDI-prediction-KG-embeddings-Conv-LSTM)**|
|**2019-07-31**|**Attention-Wrapped Hierarchical BLSTMs for DDI Extraction**|Vahab Mostafapour et.al.|[1907.13561](http://arxiv.org/abs/1907.13561)|null|
|**2020-02-20**|**Graph Embedding on Biomedical Networks: Methods, Applications, and Evaluations**|Xiang Yue et.al.|[1906.05017](http://arxiv.org/abs/1906.05017)|**[link](https://github.com/xiangyue9607/BioNEV)**|
|**2019-05-17**|**A Multi-Task Learning Framework for Extracting Drugs and Their Interactions from Drug Labels**|Tung Tran et.al.|[1905.07464](http://arxiv.org/abs/1905.07464)|null|
|**2019-05-02**|**Drug-Drug Adverse Effect Prediction with Graph Co-Attention**|Andreea Deac et.al.|[1905.00534](http://arxiv.org/abs/1905.00534)|null|
|**2019-08-05**|**Detecting drug-drug interactions using artificial neural networks and classic graph similarity measures**|Guy Shtar et.al.|[1903.04571](http://arxiv.org/abs/1903.04571)|**[link](https://github.com/goolig/DDI_prediction)**|
|**2019-02-22**|**Drug-drug interaction prediction based on co-medication patterns and graph matching**|Wen-Hao Chiang et.al.|[1902.08675](http://arxiv.org/abs/1902.08675)|null|
|**2019-01-18**|**Exploring Semi-supervised Variational Autoencoders for Biomedical Relation Extraction**|Yijia Zhang et.al.|[1901.06103](http://arxiv.org/abs/1901.06103)|null|
|**2018-11-01**|**Multi-Label Robust Factorization Autoencoder and its Application in Predicting Drug-Drug Interactions**|Xu Chu et.al.|[1811.00208](http://arxiv.org/abs/1811.00208)|null|
|**2019-03-07**|**GAMENet: Graph Augmented MEmory Networks for Recommending Medication Combination**|Junyuan Shang et.al.|[1809.01852](http://arxiv.org/abs/1809.01852)|**[link](https://github.com/sjy1203/GAMENet)**|
|**2018-06-14**|**Natural Language Processing for EHR-Based Computational Phenotyping**|Zexian Zeng et.al.|[1806.04820](http://arxiv.org/abs/1806.04820)|null|
|**2018-05-15**|**Enhancing Drug-Drug Interaction Extraction from Texts by Molecular Structure Information**|Masaki Asada et.al.|[1805.05593](http://arxiv.org/abs/1805.05593)|null|
|**2020-01-02**|**City-wide Analysis of Electronic Health Records Reveals Gender and Age Biases in the Administration of Known Drug-Drug Interactions**|Rion Brattig Correia et.al.|[1803.03571](http://arxiv.org/abs/1803.03571)|**[link](https://github.com/rionbr/DDIBlumenau)**|
|**2018-03-08**|**Drug Recommendation toward Safe Polypharmacy**|Wen-Hao Chiang et.al.|[1803.03185](http://arxiv.org/abs/1803.03185)|null|
|**2018-04-26**|**Modeling polypharmacy side effects with graph convolutional networks**|Marinka Zitnik et.al.|[1802.00543](http://arxiv.org/abs/1802.00543)|**[link](https://github.com/mims-harvard/decagon)**|
|**2018-03-12**|**Predicting Rich Drug-Drug Interactions via Biomedical Knowledge Graphs and Text Jointly Embedding**|Meng Wang et.al.|[1712.08875](http://arxiv.org/abs/1712.08875)|null|
|**2020-11-29**|**SMR: Medical Knowledge Graph Embedding for Safe Medicine Recommendation**|Fang Gong et.al.|[1710.05980](http://arxiv.org/abs/1710.05980)|null|
|**2017-05-23**|**An effective algorithm for hyperparameter optimization of neural networks**|Gonzalo Diaz et.al.|[1705.08520](http://arxiv.org/abs/1705.08520)|null|
|**2017-05-18**|**Drug-drug Interaction Extraction via Recurrent Neural Network with Multiple Attention Layers**|Zibo Yi et.al.|[1705.03261](http://arxiv.org/abs/1705.03261)|null|
|**2017-02-11**|**Automated Identification of Drug-Drug Interactions in Pediatric Congestive Heart Failure Patients**|Daniel Miller et.al.|[1702.04615](http://arxiv.org/abs/1702.04615)|null|
|**2017-08-13**|**Drug-Drug Interaction Extraction from Biomedical Text Using Long Short Term Memory Network**|Sunil Kumar Sahu et.al.|[1701.08303](http://arxiv.org/abs/1701.08303)|**[link](https://github.com/sunilitggu/DDI-extraction-through-LSTM)**|
|**2015-12-21**|**A mathematical model of granulopoiesis incorporating the negative feedback dynamics and kinetics of G-CSF/neutrophil binding and internalisation**|Morgan Craig et.al.|[1512.06896](http://arxiv.org/abs/1512.06896)|null|
|**2016-01-14**|**Monitoring Potential Drug Interactions and Reactions via Network Analysis of Instagram User Timelines**|Rion Brattig Correia et.al.|[1510.01006](http://arxiv.org/abs/1510.01006)|null|
|**2015-07-20**|**Provenance-Centered Dataset of Drug-Drug Interactions**|Juan M. Banda et.al.|[1507.05408](http://arxiv.org/abs/1507.05408)|null|

<p align=right>(<a href=#updated-on-20240320>back to top</a>)</p>

## MG

|Publish Date|Title|Authors|PDF|Code|
|---|---|---|---|---|
|**2006-12-18**|**Quantum state transfer from light to molecules via coherent two-color photo-association in an atomic Bose-Einstein condensate**|Hui Jing et.al.|[quant-ph/0512149](http://arxiv.org/abs/quant-ph/0512149)|null|
|**2024-03-06**|**Sculpting Molecules in 3D: A Flexible Substructure Aware Framework for Text-Oriented Molecular Optimization**|Kaiwei Zhang et.al.|[2403.03425](http://arxiv.org/abs/2403.03425)|null|
|**2024-01-15**|**Binding-Adaptive Diffusion Models for Structure-Based Drug Design**|Zhilin Huang et.al.|[2402.18583](http://arxiv.org/abs/2402.18583)|**[link](https://github.com/yangling0818/binddm)**|
|**2024-01-18**|**Attention Based Molecule Generation via Hierarchical Variational Autoencoder**|Divahar Sivanesan et.al.|[2402.16854](http://arxiv.org/abs/2402.16854)|null|
|**2024-03-15**|**Structure-Based Drug Design via 3D Molecular Generative Pre-training and Sampling**|Yuwei Yang et.al.|[2402.14315](http://arxiv.org/abs/2402.14315)|null|
|**2024-02-13**|**Zero Shot Molecular Generation via Similarity Kernels**|Rokas Elijošius et.al.|[2402.08708](http://arxiv.org/abs/2402.08708)|**[link](https://github.com/zincware/zndraw)**|
|**2024-02-10**|**ChemLLM: A Chemical Large Language Model**|Di Zhang et.al.|[2402.06852](http://arxiv.org/abs/2402.06852)|null|
|**2023-12-28**|**Hybrid quantum cycle generative adversarial network for small molecule generation**|Matvei Anoshin et.al.|[2402.00014](http://arxiv.org/abs/2402.00014)|null|
|**2023-09-24**|**Curiosity as a Self-Supervised Method to Improve Exploration in De novo Drug Design**|Mohamed-Amine Chadi et.al.|[2401.06771](http://arxiv.org/abs/2401.06771)|**[link](https://github.com/amine179/curiosity-rl-for-drug-design)**|
|**2023-12-21**|**De novo Drug Design using Reinforcement Learning with Multiple GPT Agents**|Xiuyuan Hu et.al.|[2401.06155](http://arxiv.org/abs/2401.06155)|**[link](https://github.com/hxyfighter/molrl-mgpt)**|
|**2023-11-21**|**nach0: Multimodal Natural and Chemical Languages Foundation Model**|Micha Livne et.al.|[2311.12410](http://arxiv.org/abs/2311.12410)|null|
|**2023-11-02**|**Investigating the Behavior of Diffusion Models for Accelerating Electronic Structure Calculations**|Daniel Rothchild et.al.|[2311.01491](http://arxiv.org/abs/2311.01491)|null|
|**2023-12-20**|**TacoGFN: Target Conditioned GFlowNet for Structure-Based Drug Design**|Tony Shen et.al.|[2310.03223](http://arxiv.org/abs/2310.03223)|null|
|**2024-02-06**|**Drug Discovery with Dynamic Goal-aware Fragments**|Seul Lee et.al.|[2310.00841](http://arxiv.org/abs/2310.00841)|null|
|**2023-09-28**|**Compositional Sculpting of Iterative Generative Processes**|Timur Garipov et.al.|[2309.16115](http://arxiv.org/abs/2309.16115)|null|
|**2024-03-03**|**Beam Enumeration: Probabilistic Explainability For Sample Efficient Self-conditioned Molecular Design**|Jeff Guo et.al.|[2309.13957](http://arxiv.org/abs/2309.13957)|**[link](https://github.com/schwallergroup/augmented_memory)**|
|**2023-12-04**|**ChemSpaceAL: An Efficient Active Learning Methodology Applied to Protein-Specific Molecular Generation**|Gregory W. Kyro et.al.|[2309.05853](http://arxiv.org/abs/2309.05853)|**[link](https://github.com/batistagroup/chemspaceal)**|
|**2024-02-20**|**Will More Expressive Graph Neural Networks do Better on Generative Tasks?**|Xiandong Zou et.al.|[2308.11978](http://arxiv.org/abs/2308.11978)|null|
|**2023-11-28**|**PoseBusters: AI-based docking methods fail to generate physically valid poses or generalise to novel sequences**|Martin Buttenschoen et.al.|[2308.05777](http://arxiv.org/abs/2308.05777)|**[link](https://github.com/maabuu/posebusters)**|
|**2023-08-04**|**Delete: Deep Lead Optimization Enveloped in Protein Pocket through Unified Deleting Strategies and a Structure-aware Network**|Haotian Zhang et.al.|[2308.02172](http://arxiv.org/abs/2308.02172)|null|
|**2023-06-13**|**Multi-objective Molecular Optimization for Opioid Use Disorder Treatment Using Generative Network Complex**|Hongsong Feng et.al.|[2306.07484](http://arxiv.org/abs/2306.07484)|null|
|**2023-06-29**|**Goal-conditioned GFlowNets for Controllable Multi-Objective Molecular Design**|Julien Roy et.al.|[2306.04620](http://arxiv.org/abs/2306.04620)|null|
|**2023-05-10**|**Augmented Memory: Capitalizing on Experience Replay to Accelerate De Novo Molecular Design**|Jeff Guo et.al.|[2305.16160](http://arxiv.org/abs/2305.16160)|null|
|**2023-05-22**|**Learning Subpocket Prototypes for Generalizable Structure-based Drug Design**|Zaixi Zhang et.al.|[2305.13997](http://arxiv.org/abs/2305.13997)|**[link](https://github.com/zaixizhang/flag)**|
|**2023-05-26**|**MolXPT: Wrapping Molecules with Text for Generative Pre-training**|Zequn Liu et.al.|[2305.10688](http://arxiv.org/abs/2305.10688)|null|
|**2023-02-14**|**PrefixMol: Target- and Chemistry-aware Molecule Design via Prefix Embedding**|Zhangyang Gao et.al.|[2302.07120](http://arxiv.org/abs/2302.07120)|null|
|**2023-02-12**|**3D Molecular Generation via Virtual Dynamics**|Shuqi Lu et.al.|[2302.05847](http://arxiv.org/abs/2302.05847)|null|
|**2023-02-26**|**De Novo Molecular Generation via Connection-aware Motif Mining**|Zijie Geng et.al.|[2302.01129](http://arxiv.org/abs/2302.01129)|**[link](https://github.com/miralab-ustc/ai4sci-micam)**|
|**2024-03-04**|**Domain-Agnostic Molecular Generation with Chemical Feedback**|Yin Fang et.al.|[2301.11259](http://arxiv.org/abs/2301.11259)|**[link](https://github.com/zjunlp/MolGen)**|
|**2023-01-12**|**Multi-Constraint Molecular Generation using Sparsely Labelled Training Data for Localized High-Concentration Electrolyte Diluent Screening**|Jonathan P. Mailoa et.al.|[2301.04814](http://arxiv.org/abs/2301.04814)|null|
|**2022-11-23**|**Group SELFIES: A Robust Fragment-Based Molecular String Representation**|Austin Cheng et.al.|[2211.13322](http://arxiv.org/abs/2211.13322)|**[link](https://github.com/aspuru-guzik-group/group-selfies)**|
|**2022-11-16**|**Molecular Fingerprints for Robust and Efficient ML-Driven Molecular Generation**|Ruslan N. Tazhigulov et.al.|[2211.09086](http://arxiv.org/abs/2211.09086)|null|
|**2022-10-13**|**Modular Flows: Differential Molecular Generation**|Yogesh Verma et.al.|[2210.06032](http://arxiv.org/abs/2210.06032)|null|
|**2023-07-02**|**The Vendi Score: A Diversity Evaluation Metric for Machine Learning**|Dan Friedman et.al.|[2210.02410](http://arxiv.org/abs/2210.02410)|**[link](https://github.com/vertaix/vendi-score)**|
|**2022-08-14**|**Widely Used and Fast De Novo Drug Design by a Protein Sequence-Based Reinforcement Learning Model**|Yaqin Li et.al.|[2209.07405](http://arxiv.org/abs/2209.07405)|null|
|**2022-09-02**|**Exploiting Pretrained Biochemical Language Models for Targeted Drug Design**|Gökçe Uludoğan et.al.|[2209.00981](http://arxiv.org/abs/2209.00981)|**[link](https://github.com/boun-tabi/biochemical-lms-for-drug-design)**|
|**2022-08-23**|**String-based Molecule Generation via Multi-decoder VAE**|Kisoo Kwon et.al.|[2208.10718](http://arxiv.org/abs/2208.10718)|null|
|**2022-10-27**|**A biologically-inspired multi-modal evaluation of molecular generative machine learning**|Elizaveta Vinogradova et.al.|[2208.09658](http://arxiv.org/abs/2208.09658)|null|
|**2022-07-27**|**Atomic structure generation from reconstructing structural fingerprints**|Victor Fung et.al.|[2207.13227](http://arxiv.org/abs/2207.13227)|**[link](https://github.com/fung-lab/structrepgen)**|
|**2022-07-23**|**A Ligand-and-structure Dual-driven Deep Learning Method for the Discovery of Highly Potent GnRH1R Antagonist to treat Uterine Diseases**|Song Li et.al.|[2207.11547](http://arxiv.org/abs/2207.11547)|null|
|**2022-07-02**|**PGMG: A Pharmacophore-Guided Deep Learning Approach for Bioactive Molecular Generation**|Huimin Zhu et.al.|[2207.00821](http://arxiv.org/abs/2207.00821)|**[link](https://github.com/csubiogroup/pgmg)**|
|**2022-06-23**|**Similarity-aware Positive Instance Sampling for Graph Contrastive Pre-training**|Xueyi Liu et.al.|[2206.11959](http://arxiv.org/abs/2206.11959)|null|
|**2023-06-03**|**Exploring Chemical Space with Score-based Out-of-distribution Generation**|Seul Lee et.al.|[2206.07632](http://arxiv.org/abs/2206.07632)|**[link](https://github.com/seullee05/mood)**|
|**2023-03-06**|**An Unpooling Layer for Graph Generation**|Yinglong Guo et.al.|[2206.01874](http://arxiv.org/abs/2206.01874)|**[link](https://github.com/guo00413/graph_unpooling)**|
|**2022-05-26**|**Protein Structure and Sequence Generation with Equivariant Denoising Diffusion Probabilistic Models**|Namrata Anand et.al.|[2205.15019](http://arxiv.org/abs/2205.15019)|null|
|**2022-05-01**|**Conditional $β$ -VAE for De Novo Molecular Generation**|Ryan J Richards et.al.|[2205.01592](http://arxiv.org/abs/2205.01592)|null|
|**2022-06-16**|**Equivariant Diffusion for Molecule Generation in 3D**|Emiel Hoogeboom et.al.|[2203.17003](http://arxiv.org/abs/2203.17003)|**[link](https://github.com/ehoogeboom/e3_diffusion_for_molecules)**|
|**2022-03-15**|**Data-Efficient Graph Grammar Learning for Molecular Generation**|Minghao Guo et.al.|[2203.08031](http://arxiv.org/abs/2203.08031)|**[link](https://github.com/gmh14/data_efficient_grammar)**|
|**2022-02-14**|**Disentangle VAE for Molecular Generation**|Yanbo Wang et.al.|[2202.06794](http://arxiv.org/abs/2202.06794)|null|
|**2022-10-21**|**Target-aware Molecular Graph Generation**|Cheng Tan et.al.|[2202.04829](http://arxiv.org/abs/2202.04829)|null|
|**2022-01-28**|**FastFlows: Flow-Based Models for Molecular Graph Generation**|Nathan C. Frey et.al.|[2201.12419](http://arxiv.org/abs/2201.12419)|**[link](https://github.com/aspuru-guzik-group/selfies)**|
|**2022-05-19**|**Improving VAE based molecular representations for compound property prediction**|A. Tevosyan et.al.|[2201.04929](http://arxiv.org/abs/2201.04929)|**[link](https://github.com/znavoyan/vae-embeddings)**|
|**2021-12-16**|**A molecular generative model with genetic algorithm and tree search for cancer samples**|Sejin Park et.al.|[2112.08959](http://arxiv.org/abs/2112.08959)|null|
|**2022-11-16**|**HelixMO: Sample-Efficient Molecular Optimization in Scene-Sensitive Latent Space**|Zhiyuan Chen et.al.|[2112.00905](http://arxiv.org/abs/2112.00905)|null|
|**2021-11-25**|**Fragment-based molecular generative model with high generalization ability and synthetic accessibility**|Seonghwan Seo et.al.|[2111.12907](http://arxiv.org/abs/2111.12907)|**[link](https://github.com/jaechang-hits/bbar-pytorch)**|
|**2021-10-24**|**De Novo Molecular Generation with Stacked Adversarial Model**|Yuansan Liu et.al.|[2110.12454](http://arxiv.org/abs/2110.12454)|null|
|**2021-10-16**|**Motif-based Graph Self-Supervised Learning for Molecular Property Prediction**|Zaixi Zhang et.al.|[2110.00987](http://arxiv.org/abs/2110.00987)|**[link](https://github.com/zaixizhang/MGSSL)**|
|**2021-08-06**|**Molecule Generation Experience: An Open Platform of Material Design for Public Users**|Seiji Takeda et.al.|[2108.03044](http://arxiv.org/abs/2108.03044)|null|
|**2021-06-30**|**Improving black-box optimization in VAE latent space using decoder uncertainty**|Pascal Notin et.al.|[2107.00096](http://arxiv.org/abs/2107.00096)|null|
|**2021-02-03**|**MolGrow: A Graph Normalizing Flow for Hierarchical Molecular Generation**|Maksim Kuznetsov et.al.|[2106.05856](http://arxiv.org/abs/2106.05856)|null|
|**2022-06-28**|**Multiresolution Equivariant Graph Variational Autoencoder**|Truong Son Hy et.al.|[2106.00967](http://arxiv.org/abs/2106.00967)|**[link](https://github.com/hytruongson/mgvae)**|
|**2021-04-09**|**A Graph VAE and Graph Transformer Approach to Generating Molecular Graphs**|Joshua Mitton et.al.|[2104.04345](http://arxiv.org/abs/2104.04345)|null|
|**2021-02-09**|**Graph Energy-based Model for Substructure Preserving Molecular Design**|Ryuichiro Hataya et.al.|[2102.04600](http://arxiv.org/abs/2102.04600)|null|
|**2021-05-27**|**Molecular graph generation with Graph Neural Networks**|Pietro Bongini et.al.|[2012.07397](http://arxiv.org/abs/2012.07397)|null|
|**2021-02-11**|**Physics-Constrained Predictive Molecular Latent Space Discovery with Graph Scattering Variational Autoencoder**|Navid Shervani-Tabar et.al.|[2009.13878](http://arxiv.org/abs/2009.13878)|**[link](https://github.com/zabaras/GSVAE)**|
|**2020-10-05**|**Scaffold-constrained molecular generation**|Maxime Langevin et.al.|[2009.07778](http://arxiv.org/abs/2009.07778)|null|
|**2020-07-20**|**Visualizing Deep Graph Generative Models for Drug Discovery**|Karan Yang et.al.|[2007.10333](http://arxiv.org/abs/2007.10333)|null|
|**2020-07-10**|**DC electricity generation from dynamic polarized water-semiconductor interface**|Yanfei Yan et.al.|[2007.04556](http://arxiv.org/abs/2007.04556)|null|
|**2022-02-09**|**GEOM: Energy-annotated molecular conformations for property prediction and molecular generation**|Simon Axelrod et.al.|[2006.05531](http://arxiv.org/abs/2006.05531)|**[link](https://github.com/learningmatter-mit/geom)**|
|**2020-07-06**|**PaccMann $^{RL}$ on SARS-CoV-2: Designing antiviral candidates with conditional generative models**|Jannis Born et.al.|[2005.13285](http://arxiv.org/abs/2005.13285)|**[link](https://github.com/PaccMann/paccmann_sarscov2)**|
|**2020-03-04**|**Deterministic Decoding for Discrete Data in Variational Autoencoders**|Daniil Polykovskiy et.al.|[2003.02174](http://arxiv.org/abs/2003.02174)|**[link](https://github.com/insilicomedicine/DD-VAE)**|
|**2020-02-17**|**The Synthesizability of Molecules Proposed by Generative Models**|Wenhao Gao et.al.|[2002.07007](http://arxiv.org/abs/2002.07007)|**[link](https://github.com/wenhao-gao/askcos_synthesizability)**|
|**2019-11-13**|**Molecular Generative Model Based On Adversarially Regularized Autoencoder**|Seung Hwan Hong et.al.|[1912.05617](http://arxiv.org/abs/1912.05617)|**[link](https://github.com/gicsaw/ARAE_SMILES)**|
|**2019-09-05**|**DeepScaffold: a comprehensive tool for scaffold-based de novo drug discovery using deep learning**|Yibo Li et.al.|[1908.07209](http://arxiv.org/abs/1908.07209)|null|
|**2019-07-02**|**Generative Models for Automatic Chemical Design**|Daniel Schwalbe-Koda et.al.|[1907.01632](http://arxiv.org/abs/1907.01632)|null|
|**2019-05-31**|**Scaffold-based molecular design using graph generative model**|Jaechang Lim et.al.|[1905.13639](http://arxiv.org/abs/1905.13639)|null|
|**2020-10-28**|**Molecular Sets (MOSES): A Benchmarking Platform for Molecular Generation Models**|Daniil Polykovskiy et.al.|[1811.12823](http://arxiv.org/abs/1811.12823)|**[link](https://github.com/molecularsets/moses)**|
|**2018-06-15**|**Molecular generative model based on conditional variational autoencoder for de novo molecular design**|Jaechang Lim et.al.|[1806.05805](http://arxiv.org/abs/1806.05805)|null|
|**2019-03-29**|**Junction Tree Variational Autoencoder for Molecular Graph Generation**|Wengong Jin et.al.|[1802.04364](http://arxiv.org/abs/1802.04364)|**[link](https://github.com/wengong-jin/icml18-jtnn)**|
|**2017-09-29**|**ChemTS: An Efficient Python Library for de novo Molecular Generation**|Xiufeng Yang et.al.|[1710.00616](http://arxiv.org/abs/1710.00616)|**[link](https://github.com/tsudalab/ChemTS)**|
|**2017-05-17**|**Molecular Generation with Recurrent Neural Networks (RNNs)**|Esben Jannik Bjerrum et.al.|[1705.04612](http://arxiv.org/abs/1705.04612)|null|
|**2017-10-16**|**Sequence Tutor: Conservative Fine-Tuning of Sequence Generation Models with KL-control**|Natasha Jaques et.al.|[1611.02796](http://arxiv.org/abs/1611.02796)|null|

<p align=right>(<a href=#updated-on-20240320>back to top</a>)</p>

## MPP

|Publish Date|Title|Authors|PDF|Code|
|---|---|---|---|---|
|**2024-02-21**|**BBA: Bi-Modal Behavioral Alignment for Reasoning with Large Vision-Language Models**|Xueliang Zhao et.al.|[2402.13577](http://arxiv.org/abs/2402.13577)|null|
|**2024-02-20**|**Equivariant Pretrained Transformer for Unified Geometric Learning on Multi-Domain 3D Molecules**|Rui Jiao et.al.|[2402.12714](http://arxiv.org/abs/2402.12714)|null|
|**2024-02-18**|**The Impact of Domain Knowledge and Multi-Modality on Intelligent Molecular Property Prediction: A Systematic Survey**|Taojie Kuang et.al.|[2402.07249](http://arxiv.org/abs/2402.07249)|null|
|**2024-02-10**|**ChemLLM: A Chemical Large Language Model**|Di Zhang et.al.|[2402.06852](http://arxiv.org/abs/2402.06852)|null|
|**2024-02-07**|**Triplet Interaction Improves Graph Transformers: Accurate Molecular Graph Learning with Triplet Graph Transformers**|Md Shamim Hussain et.al.|[2402.04538](http://arxiv.org/abs/2402.04538)|**[link](https://github.com/shamim-hussain/tgt)**|
|**2024-02-03**|**Structure-Aware E(3)-Invariant Molecular Conformer Aggregation Networks**|Duy M. H. Nguyen et.al.|[2402.01975](http://arxiv.org/abs/2402.01975)|null|
|**2024-02-02**|**Graph Multi-Similarity Learning for Molecular Property Prediction**|Hao Xu et.al.|[2401.17615](http://arxiv.org/abs/2401.17615)|null|
|**2024-01-30**|**MolPLA: A Molecular Pretraining Framework for Learning Cores, R-Groups and their Linker Joints**|Mogan Gim et.al.|[2401.16771](http://arxiv.org/abs/2401.16771)|**[link](https://github.com/dmis-lab/molpla)**|
|**2024-01-29**|**Enhancing Molecular Property Prediction with Auxiliary Learning and Task-Specific Adaptation**|Vishal Dey et.al.|[2401.16299](http://arxiv.org/abs/2401.16299)|**[link](https://github.com/vishaldeyiiest/graphta)**|
|**2023-12-28**|**Adjustable Molecular Representation for Unified Pre-training Strategy**|Yan Ding et.al.|[2401.06166](http://arxiv.org/abs/2401.06166)|null|
|**2024-01-30**|**TwinBooster: Synergising Large Language Models with Barlow Twins and Gradient Boosting for Enhanced Molecular Property Prediction**|Maximilian G. Schuh et.al.|[2401.04478](http://arxiv.org/abs/2401.04478)|**[link](https://github.com/maxischuh/twinbooster)**|
|**2024-01-09**|**Multi-Modal Representation Learning for Molecular Property Prediction: Sequence, Graph, Geometry**|Zeyu Wang et.al.|[2401.03369](http://arxiv.org/abs/2401.03369)|**[link](https://github.com/vencent-won/sggrl)**|
|**2023-12-29**|**Integrating Chemical Language and Molecular Graph in Multimodal Fused Deep Learning for Drug Property Prediction**|Xiaohua Lu et.al.|[2312.17495](http://arxiv.org/abs/2312.17495)|null|
|**2023-12-28**|**Molecular Property Prediction Based on Graph Structure Learning**|Bangyi Zhao et.al.|[2312.16855](http://arxiv.org/abs/2312.16855)|null|
|**2024-01-18**|**Pre-training of Molecular GNNs via Conditional Boltzmann Generator**|Daiki Koge et.al.|[2312.13110](http://arxiv.org/abs/2312.13110)|null|
|**2023-12-19**|**Accurate harmonic vibrational frequencies for diatomic molecules via quantum computing**|Shih-Kai Chou et.al.|[2312.12320](http://arxiv.org/abs/2312.12320)|null|
|**2023-12-12**|**SE(3)-Invariant Multiparameter Persistent Homology for Chiral-Sensitive Molecular Property Prediction**|Andac Demir et.al.|[2312.07633](http://arxiv.org/abs/2312.07633)|null|
|**2023-12-06**|**Enhancing Molecular Property Prediction via Mixture of Collaborative Experts**|Xu Yao et.al.|[2312.03292](http://arxiv.org/abs/2312.03292)|**[link](https://github.com/Hyacinth-YX/mixture-of-collaborative-experts)**|
|**2024-01-15**|**Provable Adversarial Robustness for Group Equivariant Tasks: Graphs, Point Clouds, Molecules, and More**|Jan Schuchardt et.al.|[2312.02708](http://arxiv.org/abs/2312.02708)|null|
|**2023-12-01**|**Removing Biases from Molecular Representations via Information Maximization**|Chenyu Wang et.al.|[2312.00718](http://arxiv.org/abs/2312.00718)|**[link](https://github.com/uhlerlab/infocore)**|
|**2023-11-29**|**Improving Self-supervised Molecular Representation Learning using Persistent Homology**|Yuankai Luo et.al.|[2311.17327](http://arxiv.org/abs/2311.17327)|**[link](https://github.com/luoyk1999/molecular-homology)**|
|**2023-11-28**|**MultiModal-Learning for Predicting Molecular Properties: A Framework Based on Image and Graph Structures**|Zhuoyuan Wang et.al.|[2311.16666](http://arxiv.org/abs/2311.16666)|null|
|**2023-11-17**|**Multiparameter Persistent Homology for Molecular Property Prediction**|Andac Demir et.al.|[2311.10808](http://arxiv.org/abs/2311.10808)|null|
|**2023-11-08**|**Identifying Semantic Component for Robust Molecular Property Prediction**|Zijian Li et.al.|[2311.04837](http://arxiv.org/abs/2311.04837)|**[link](https://github.com/dmirlab-group/sci)**|
|**2023-11-05**|**From molecules to scaffolds to functional groups: building context-dependent molecular representation via multi-channel learning**|Yue Wan et.al.|[2311.02798](http://arxiv.org/abs/2311.02798)|null|
|**2023-11-03**|**Sliced Denoising: A Physics-Informed Molecular Pre-Training Method**|Yuyan Ni et.al.|[2311.02124](http://arxiv.org/abs/2311.02124)|null|
|**2023-10-25**|**Unsupervised Learning of Molecular Embeddings for Enhanced Clustering and Emergent Properties for Chemical Compounds**|Jaiveer Gill et.al.|[2310.18367](http://arxiv.org/abs/2310.18367)|null|
|**2023-10-26**|**Sliceformer: Make Multi-head Attention as Simple as Sorting in Discriminative Tasks**|Shen Yuan et.al.|[2310.17683](http://arxiv.org/abs/2310.17683)|**[link](https://github.com/sds-lab/sliceformer)**|
|**2023-10-22**|**UniMAP: Universal SMILES-Graph Representation Learning**|Shikun Feng et.al.|[2310.14216](http://arxiv.org/abs/2310.14216)|**[link](https://github.com/fengshikun/unimap)**|
|**2023-10-20**|**Improving Molecular Properties Prediction Through Latent Space Fusion**|Eduardo Soares et.al.|[2310.13802](http://arxiv.org/abs/2310.13802)|**[link](https://github.com/ibm/molformer)**|
|**2023-10-13**|**In-Context Learning for Few-Shot Molecular Property Prediction**|Christopher Fifty et.al.|[2310.08863](http://arxiv.org/abs/2310.08863)|null|
|**2023-10-11**|**Atom-Motif Contrastive Transformer for Molecular Property Prediction**|Wentao Yu et.al.|[2310.07351](http://arxiv.org/abs/2310.07351)|null|
|**2023-10-10**|**Lo-Hi: Practical ML Drug Discovery Benchmark**|Simon Steshin et.al.|[2310.06399](http://arxiv.org/abs/2310.06399)|null|
|**2023-10-08**|**PointGAT: A quantum chemical property prediction model integrating graph attention and 3D geometry**|Rong Zhang et.al.|[2310.05217](http://arxiv.org/abs/2310.05217)|null|
|**2023-10-10**|**GPT-MolBERTa: GPT Molecular Features Language Model for molecular property prediction**|Suryanarayanan Balaji et.al.|[2310.03030](http://arxiv.org/abs/2310.03030)|null|
|**2023-08-25**|**Synergistic Fusion of Graph and Transformer Features for Enhanced Molecular Property Prediction**|M V Sai Prakash et.al.|[2310.03027](http://arxiv.org/abs/2310.03027)|null|
|**2023-10-04**|**On the Stability of Expressive Positional Encodings for Graph Neural Networks**|Yinan Huang et.al.|[2310.02579](http://arxiv.org/abs/2310.02579)|**[link](https://github.com/Graph-COM/SPE)**|
|**2023-10-01**|**Hierarchical Adaptation with Hypernetworks for Few-shot Molecular Property Prediction**|Shiguang Wu et.al.|[2310.00614](http://arxiv.org/abs/2310.00614)|null|
|**2024-01-18**|**3D-Mol: A Novel Contrastive Learning Framework for Molecular Property Prediction with 3D Information**|Taojie Kuang et.al.|[2309.17366](http://arxiv.org/abs/2309.17366)|null|
|**2023-09-08**|**Motif-aware Attribute Masking for Molecular Graph Pre-training**|Eric Inae et.al.|[2309.04589](http://arxiv.org/abs/2309.04589)|**[link](https://github.com/einae-nd/moama-dev)**|
|**2023-09-08**|**3D Denoisers are Good 2D Teachers: Molecular Pretraining via Denoising and Cross-Modal Distillation**|Sungjun Cho et.al.|[2309.04062](http://arxiv.org/abs/2309.04062)|null|
|**2023-09-06**|**Using Multiple Vector Channels Improves E(n)-Equivariant Graph Neural Networks**|Daniel Levy et.al.|[2309.03139](http://arxiv.org/abs/2309.03139)|null|
|**2023-09-04**|**Hierarchical Grammar-Induced Geometry for Data-Efficient Molecular Property Prediction**|Minghao Guo et.al.|[2309.01788](http://arxiv.org/abs/2309.01788)|**[link](https://github.com/gmh14/geo-deg)**|
|**2023-09-01**|**Geometry-aware Line Graph Transformer Pre-training for Molecular Property Prediction**|Peizhen Bai et.al.|[2309.00483](http://arxiv.org/abs/2309.00483)|null|
|**2023-08-17**|**On Data Imbalance in Molecular Property Prediction with Pre-training**|Limin Wang et.al.|[2308.08934](http://arxiv.org/abs/2308.08934)|null|
|**2023-08-16**|**Is Self-Supervised Pretraining Good for Extrapolation in Molecular Property Prediction?**|Shun Takashige et.al.|[2308.08129](http://arxiv.org/abs/2308.08129)|null|
|**2023-07-27**|**Rotation-Invariant Random Features Provide a Strong Baseline for Machine Learning on 3D Point Clouds**|Owen Melia et.al.|[2308.06271](http://arxiv.org/abs/2308.06271)|**[link](https://github.com/meliao/rotation-invariant-random-features)**|
|**2023-07-19**|**Uncertainty Quantification for Molecular Property Predictions with Graph Neural Architecture Search**|Shengli Jiang et.al.|[2307.10438](http://arxiv.org/abs/2307.10438)|null|
|**2023-07-14**|**Can Large Language Models Empower Molecular Property Prediction?**|Chen Qian et.al.|[2307.07443](http://arxiv.org/abs/2307.07443)|**[link](https://github.com/chnq/llm4mol)**|
|**2024-03-07**|**Predicting small molecules solubilities on endpoint devices using deep ensemble neural networks**|Mayk Caldas Ramos et.al.|[2307.05318](http://arxiv.org/abs/2307.05318)|**[link](https://github.com/ur-whitelab/mol.dev)**|
|**2023-07-10**|**ChiENN: Embracing Molecular Chirality with Graph Neural Networks**|Piotr Gaiński et.al.|[2307.02198](http://arxiv.org/abs/2307.02198)|**[link](https://github.com/gmum/chienn)**|
|**2023-08-22**|**Variational Autoencoding Molecular Graphs with Denoising Diffusion Probabilistic Model**|Daiki Koge et.al.|[2307.00623](http://arxiv.org/abs/2307.00623)|null|
|**2023-06-30**|**Why Deep Models Often cannot Beat Non-deep Counterparts on Molecular Property Prediction?**|Jun Xia et.al.|[2306.17702](http://arxiv.org/abs/2306.17702)|null|
|**2023-06-29**|**Graph Sampling-based Meta-Learning for Molecular Property Prediction**|Xiang Zhuang et.al.|[2306.16780](http://arxiv.org/abs/2306.16780)|**[link](https://github.com/hicai-zju/gs-meta)**|
|**2023-06-22**|**Molecular geometric deep learning**|Cong Shen et.al.|[2306.15065](http://arxiv.org/abs/2306.15065)|**[link](https://github.com/cs-bio/mol-gdl)**|
|**2023-06-22**|**Beyond Chemical Language: A Multimodal Approach to Enhance Molecular Property Prediction**|Eduardo Soares et.al.|[2306.14919](http://arxiv.org/abs/2306.14919)|null|
|**2023-10-28**|**Accelerating Molecular Graph Neural Networks via Knowledge Distillation**|Filip Ekström Kelvinius et.al.|[2306.14818](http://arxiv.org/abs/2306.14818)|null|
|**2023-11-13**|**Tanimoto Random Features for Scalable Molecular Machine Learning**|Austin Tripp et.al.|[2306.14809](http://arxiv.org/abs/2306.14809)|null|
|**2023-06-13**|**MolCAP: Molecular Chemical reActivity pretraining and prompted-finetuning enhanced molecular representation learning**|Yu Wang et.al.|[2306.09187](http://arxiv.org/abs/2306.09187)|null|
|**2023-06-14**|**Self-supervised Learning and Graph Classification under Heterophily**|Yilin Ding et.al.|[2306.08469](http://arxiv.org/abs/2306.08469)|null|
|**2023-07-02**|**Automated 3D Pre-Training for Molecular Property Prediction**|Xu Wang et.al.|[2306.07812](http://arxiv.org/abs/2306.07812)|**[link](https://github.com/lars-research/3d-pgt)**|
|**2024-01-20**|**Bi-level Contrastive Learning for Knowledge-Enhanced Molecule Representations**|Pengcheng Jiang et.al.|[2306.01631](http://arxiv.org/abs/2306.01631)|null|
|**2023-05-24**|**Directed Message Passing Based on Attention for Prediction of Molecular Properties**|Chen Gong et.al.|[2305.14819](http://arxiv.org/abs/2305.14819)|**[link](https://github.com/gongchen-1995/d-gats)**|
|**2023-05-22**|**Atomic and Subgraph-aware Bilateral Aggregation for Molecular Representation Learning**|Jiahao Chen et.al.|[2305.12618](http://arxiv.org/abs/2305.12618)|null|
|**2023-05-26**|**MolXPT: Wrapping Molecules with Text for Generative Pre-training**|Zequn Liu et.al.|[2305.10688](http://arxiv.org/abs/2305.10688)|null|
|**2023-06-27**|**Molecule-Morphology Contrastive Pretraining for Transferable Molecular Representation**|Cuong Q. Nguyen et.al.|[2305.09790](http://arxiv.org/abs/2305.09790)|**[link](https://github.com/GSK-AI/mocop)**|
|**2023-05-03**|**MolKD: Distilling Cross-Modal Knowledge in Chemical Reactions for Molecular Property Prediction**|Liang Zeng et.al.|[2305.01912](http://arxiv.org/abs/2305.01912)|null|
|**2023-04-24**|**Uni-QSAR: an Auto-ML Tool for Molecular Property Prediction**|Zhifeng Gao et.al.|[2304.12239](http://arxiv.org/abs/2304.12239)|null|
|**2023-04-15**|**Icospherical Chemical Objects (ICOs) allow for chemical data augmentation and maintain rotational, translation and permutation invariance**|Ella Gale et.al.|[2304.07558](http://arxiv.org/abs/2304.07558)|null|
|**2023-04-07**|**A new perspective on building efficient and expressive 3D equivariant graph neural networks**|Weitao Du et.al.|[2304.04757](http://arxiv.org/abs/2304.04757)|**[link](https://github.com/yuanqidu/leftnet)**|
|**2023-05-25**|**SELFormer: Molecular Representation Learning via SELFIES Language Models**|Atakan Yüksel et.al.|[2304.04662](http://arxiv.org/abs/2304.04662)|**[link](https://github.com/hubiodatalab/selformer)**|
|**2023-04-14**|**InstructBio: A Large-scale Semi-supervised Learning Paradigm for Biochemical Problems**|Fang Wu et.al.|[2304.03906](http://arxiv.org/abs/2304.03906)|**[link](https://github.com/smiles724/instructbio)**|
|**2023-03-27**|**HD-Bind: Encoding of Molecular Structure with Low Precision, Hyperdimensional Binary Representations**|Derek Jones et.al.|[2303.15604](http://arxiv.org/abs/2303.15604)|null|
|**2024-01-14**|**Difficulty in chirality recognition for Transformer architectures learning chemical structures from string**|Yasuhiro Yoshikai et.al.|[2303.11593](http://arxiv.org/abs/2303.11593)|**[link](https://github.com/mizuno-group/chiralitymisunderstanding)**|
|**2023-03-13**|**Molecular Property Prediction by Semantic-invariant Contrastive Learning**|Ziqiao Zhang et.al.|[2303.06902](http://arxiv.org/abs/2303.06902)|null|
|**2023-03-05**|**Deep Learning Methods for Small Molecule Drug Discovery: A Survey**|Wenhao Hu et.al.|[2303.00313](http://arxiv.org/abs/2303.00313)|null|
|**2023-03-12**|**Learning Topology-Specific Experts for Molecular Property Prediction**|Su Kim et.al.|[2302.13693](http://arxiv.org/abs/2302.13693)|**[link](https://github.com/kimsu55/toxexpert)**|
|**2023-02-09**|**GCI: A (G)raph (C)oncept (I)nterpretation Framework**|Dmitry Kazhdan et.al.|[2302.04899](http://arxiv.org/abs/2302.04899)|**[link](https://github.com/dmitrykazhdan/gci)**|
|**2023-05-12**|**GPS++: Reviving the Art of Message Passing for Molecular Property Prediction**|Dominic Masters et.al.|[2302.02947](http://arxiv.org/abs/2302.02947)|**[link](https://github.com/graphcore/ogb-lsc-pcqm4mv2)**|
|**2023-10-06**|**Implicit Geometry and Interaction Embeddings Improve Few-Shot Molecular Property Prediction**|Christopher Fifty et.al.|[2302.02055](http://arxiv.org/abs/2302.02055)|**[link](https://github.com/cfifty/ignite)**|
|**2023-01-21**|**An open unified deep graph learning framework for discovering drug leads**|Yueming Yin et.al.|[2301.03424](http://arxiv.org/abs/2301.03424)|**[link](https://github.com/yueming-yin/gafse)**|
|**2023-09-22**|**MolCPT: Molecule Continuous Prompt Tuning to Generalize Molecular Representation Learning**|Cameron Diao et.al.|[2212.10614](http://arxiv.org/abs/2212.10614)|null|
|**2023-10-27**|**Implicit Convolutional Kernels for Steerable CNNs**|Maksim Zhdanov et.al.|[2212.06096](http://arxiv.org/abs/2212.06096)|**[link](https://github.com/maxxxzdn/implicit-steerable-kernels)**|
|**2022-12-06**|**GPS++: An Optimised Hybrid MPNN/Transformer for Molecular Property Prediction**|Dominic Masters et.al.|[2212.02229](http://arxiv.org/abs/2212.02229)|**[link](https://github.com/graphcore/ogb-lsc-pcqm4mv2)**|
|**2022-11-30**|**Coordinating Cross-modal Distillation for Molecular Property Prediction**|Hao Zhang et.al.|[2211.16712](http://arxiv.org/abs/2211.16712)|null|
|**2022-11-23**|**Supervised Pretraining for Molecular Force Fields and Properties Prediction**|Xiang Gao et.al.|[2211.14429](http://arxiv.org/abs/2211.14429)|null|
|**2023-01-03**|**Invariance-Aware Randomized Smoothing Certificates**|Jan Schuchardt et.al.|[2211.14207](http://arxiv.org/abs/2211.14207)|null|
|**2023-11-06**|**BatmanNet: Bi-branch Masked Graph Transformer Autoencoder for Molecular Representation**|Zhen Wang et.al.|[2211.13979](http://arxiv.org/abs/2211.13979)|**[link](https://github.com/wz-create/batmannet)**|
|**2022-11-25**|**Extreme Acceleration of Graph Neural Network-based Prediction Models for Quantum Chemistry**|Hatem Helal et.al.|[2211.13853](http://arxiv.org/abs/2211.13853)|null|
|**2023-08-16**|**An ensemble of VisNet, Transformer-M, and pretraining models for molecular property prediction in OGB Large-Scale Challenge @ NeurIPS 2022**|Yusong Wang et.al.|[2211.12791](http://arxiv.org/abs/2211.12791)|null|
|**2022-11-20**|**Heterogenous Ensemble of Models for Molecular Property Prediction**|Sajad Darabi et.al.|[2211.11035](http://arxiv.org/abs/2211.11035)|**[link](https://github.com/jfpuget/nvidia-pcqm4mv2)**|
|**2022-11-03**|**A 3D-Shape Similarity-based Contrastive Approach to Molecular Representation Learning**|Austin Atsango et.al.|[2211.02130](http://arxiv.org/abs/2211.02130)|null|
|**2022-10-18**|**PEMP: Leveraging Physics Properties to Enhance Molecular Property Prediction**|Yuancheng Sun et.al.|[2211.01978](http://arxiv.org/abs/2211.01978)|null|
|**2023-02-25**|**Isotropic Gaussian Processes on Finite Spaces of Graphs**|Viacheslav Borovitskiy et.al.|[2211.01689](http://arxiv.org/abs/2211.01689)|**[link](https://github.com/ibm/graph_space_gps)**|
|**2023-08-16**|**ViSNet: an equivariant geometry-enhanced graph neural network with vector-scalar interactive message passing for molecules**|Yusong Wang et.al.|[2210.16518](http://arxiv.org/abs/2210.16518)|null|
|**2022-10-21**|**Multimodal Model with Text and Drug Embeddings for Adverse Drug Reaction Classification**|Andrey Sakhovskiy et.al.|[2210.13238](http://arxiv.org/abs/2210.13238)|**[link](https://github.com/andoree/smm4h_2021_classification)**|
|**2022-10-19**|**Structure-based drug design with geometric deep learning**|Clemens Isert et.al.|[2210.11250](http://arxiv.org/abs/2210.11250)|null|
|**2022-10-15**|**Substructure-Atom Cross Attention for Molecular Representation Learning**|Jiye Kim et.al.|[2210.08243](http://arxiv.org/abs/2210.08243)|null|
|**2022-12-27**|**ProGReST: Prototypical Graph Regression Soft Trees for Molecular Property Prediction**|Dawid Rymarczyk et.al.|[2210.03745](http://arxiv.org/abs/2210.03745)|**[link](https://github.com/gmum/progrest)**|
|**2022-09-29**|**Improving Molecular Pretraining with Complementary Featurizations**|Yanqiao Zhu et.al.|[2209.15101](http://arxiv.org/abs/2209.15101)|null|

<p align=right>(<a href=#updated-on-20240320>back to top</a>)</p>

## MD

|Publish Date|Title|Authors|PDF|Code|
|---|---|---|---|---|
|**2024-03-18**|**Mineral and cross-linking in collagen fibrils: The mechanical behavior of bone tissue at the nano-scale**|Julia Kamml et.al.|[2403.11753](http://arxiv.org/abs/2403.11753)|null|
|**2024-03-18**|**Surface hopping molecular dynamics simulation of ultrafast methyl iodide photodissociation mapped by Coulomb explosion imaging**|Yijue Ding et.al.|[2403.11592](http://arxiv.org/abs/2403.11592)|null|
|**2024-03-15**|**Good rates from bad coordinates: the exponential average time-dependent rate approach**|Nicodemo Mazzaferro et.al.|[2403.10668](http://arxiv.org/abs/2403.10668)|null|
|**2024-02-24**|**Coalescence of sessile aqueous droplets laden with surfactant**|S. Arbabi et.al.|[2403.10535](http://arxiv.org/abs/2403.10535)|null|
|**2024-03-15**|**Defect design in ferroelectrics -- new insights on agglomeration**|Sheng-Han Teng et.al.|[2403.10467](http://arxiv.org/abs/2403.10467)|null|
|**2024-03-15**|**Hessian-free force-gradient integrators**|Kevin Schäfers et.al.|[2403.10370](http://arxiv.org/abs/2403.10370)|null|
|**2024-03-15**|**GradNav: Accelerated Exploration of Potential Energy Surfaces with Gradient-Based Navigation**|Janghoon Ock et.al.|[2403.10358](http://arxiv.org/abs/2403.10358)|null|
|**2024-03-14**|**A general-purpose neural network potential for Ti-Al-Nb alloys towards large-scale molecular dynamics with ab initio accuracy**|Zhiqiang Zhao et.al.|[2403.09529](http://arxiv.org/abs/2403.09529)|null|
|**2024-03-13**|**Criticality in an imidazolium ionic liquid fully wetting a sapphire support**|Kevin Höllring et.al.|[2403.08449](http://arxiv.org/abs/2403.08449)|null|
|**2024-03-13**|**Influence of cholesterol on hydrogen-bond dynamics of water molecules in lipid-bilayer systems at varying temperatures**|Kokoro Shikata et.al.|[2403.08289](http://arxiv.org/abs/2403.08289)|null|
|**2024-03-13**|**Experimental observation of gapped shear waves and liquid-like to gas-like dynamical crossover in active granular matter**|Cunyuan Jiang et.al.|[2403.08285](http://arxiv.org/abs/2403.08285)|null|
|**2024-03-13**|**GCMe: Efficient implementation of Gaussian core model with smeared electrostatic interactions for molecular dynamics simulations of soft matter systems**|Benjamin Bobin Ye et.al.|[2403.08148](http://arxiv.org/abs/2403.08148)|null|
|**2024-03-11**|**Dynamics of Polyalkylfluorene Conjugated Polymers: Insights from Neutron Spectroscopy and Molecular Dynamics Simulations**|Mohamed Zbiri et.al.|[2403.06822](http://arxiv.org/abs/2403.06822)|null|
|**2024-03-12**|**Ultrafast switching of sliding ferroelectricity and dynamical magnetic field in van der Waals bilayer induced by light**|Jian Wang et.al.|[2403.06531](http://arxiv.org/abs/2403.06531)|null|
|**2024-03-09**|**A molecular dynamics simulation of the abrupt changes in the thermodynamic properties of water after formation of nano-bubbles / nano-cavities induced by passage of charged particles**|Ramin Abolfath et.al.|[2403.05880](http://arxiv.org/abs/2403.05880)|null|
|**2024-03-07**|**Electric field of DNA in solution: who is in charge?**|Jonathan G. Hedley et.al.|[2403.04949](http://arxiv.org/abs/2403.04949)|null|
|**2024-03-07**|**Induced charges in a Thomas-Fermi metal: insights from molecular simulations**|Swetha Nair et.al.|[2403.04487](http://arxiv.org/abs/2403.04487)|null|
|**2024-03-07**|**OH $^-$ ions can reduce the iodide migration in MAPI**|R. E. Brophy et.al.|[2403.04188](http://arxiv.org/abs/2403.04188)|null|
|**2024-03-06**|**Collision Cascade-Driven Evolution of Vacancy Defects in Ni-Based Concentrated Solid-Solution Alloys**|A. Aligayev et.al.|[2403.03922](http://arxiv.org/abs/2403.03922)|null|
|**2024-03-06**|**Optical and spin properties of nitrogen vacancy centers formed along the tracks of high energy heavy ions**|Wei Liu et.al.|[2403.03570](http://arxiv.org/abs/2403.03570)|null|
|**2024-03-06**|**Lattice thermal conductivity and mechanical properties of the single-layer penta-NiN2 explored by a deep-learning interatomic potential**|Pedram Mirchi et.al.|[2403.03515](http://arxiv.org/abs/2403.03515)|null|
|**2024-03-05**|**First Principles Validation of Energy Barriers in Ni $_{75}$Al$_{25}$**|Adam Fisher et.al.|[2403.03282](http://arxiv.org/abs/2403.03282)|null|
|**2024-03-04**|**Numerical Simulation of Phase Transition with the Hyperbolic Godunov-Peshkov-Romenski Model**|Pascal Mossier et.al.|[2403.01847](http://arxiv.org/abs/2403.01847)|null|
|**2024-03-02**|**Exploring Structural and Electrochemical Properties of Li $_3$TiCl$_6$ : A Machine Learning Molecular Dynamics Study**|Selva Chandrasekaran Selvaraj et.al.|[2403.01077](http://arxiv.org/abs/2403.01077)|null|
|**2024-03-01**|**Atomic scale insights into NaCl nucleation in nanoconfined environments**|Ruiyu Wang et.al.|[2403.00925](http://arxiv.org/abs/2403.00925)|null|
|**2024-03-01**|**When should PIC simulations be applied to atmospheric pressure plasmas? Impact of correlation heating**|M. Acciarri et.al.|[2403.00656](http://arxiv.org/abs/2403.00656)|null|
|**2024-02-29**|**Simulations of Classical Three-Body Thermalization in One Dimension**|M. Eltohfa et.al.|[2403.00089](http://arxiv.org/abs/2403.00089)|null|
|**2024-02-28**|**Hydrogen bonding in water under extreme confinement unveiled by nanoscale vibrational spectroscopy and simulations**|Xintong Xu et.al.|[2402.17989](http://arxiv.org/abs/2402.17989)|null|
|**2024-02-27**|**A Bayesian Committee Machine Potential for Organic Nitrogen Compounds**|Hyun Gyu Park et.al.|[2402.17132](http://arxiv.org/abs/2402.17132)|null|
|**2024-03-16**|**Virial Equation of State for a Granular System**|Subhanker Howlader et.al.|[2402.17100](http://arxiv.org/abs/2402.17100)|null|
|**2024-02-24**|**Distinguishable-particle Glassy Crystal: the simplest molecular model of glass**|Leo S. I. Lam et.al.|[2402.15805](http://arxiv.org/abs/2402.15805)|null|
|**2024-02-24**|**Prediction of Superionic State in LiH2 at Conditions Enroute to Nuclear Fusion**|Fude Li et.al.|[2402.15791](http://arxiv.org/abs/2402.15791)|null|
|**2024-02-23**|**The origin of phase separation in binary aluminosilicate glasses**|Houssam Kharouji et.al.|[2402.15314](http://arxiv.org/abs/2402.15314)|null|
|**2024-03-07**|**Vibronics of multi-material nanopillared membranes and impact on the thermal conductivity**|Lina Yang et.al.|[2402.15295](http://arxiv.org/abs/2402.15295)|null|
|**2024-02-23**|**E(n)-Equivariant Cartesian Tensor Passing Potential**|Junjie Wang et.al.|[2402.15286](http://arxiv.org/abs/2402.15286)|null|
|**2024-03-15**|**Distinguishing fission-like events from deep-inelastic collisions**|Hong Yao et.al.|[2402.15224](http://arxiv.org/abs/2402.15224)|null|
|**2024-02-23**|**Dynamics and energetics of ion adsorption at the interface between a pure ionic liquid and carbon electrodes**|Nicolas Gaudy et.al.|[2402.15199](http://arxiv.org/abs/2402.15199)|null|
|**2024-02-23**|**Friction and Memory Effects in Homogeneous Gas-Liquid Nucleation: Quest for Quantitative Rate Calculation**|Subhajit Acharya et.al.|[2402.15126](http://arxiv.org/abs/2402.15126)|null|
|**2024-02-21**|**Thermal transport in a 2D amorphous material**|Yuxi Wang et.al.|[2402.13471](http://arxiv.org/abs/2402.13471)|null|
|**2024-02-20**|**Positive temperature-dependent thermal conductivity induced by wavelike phonons in complex Ag-based argyrodites**|Niuchang Ouyang et.al.|[2402.12699](http://arxiv.org/abs/2402.12699)|null|
|**2024-02-19**|**Molecular Dynamics Simulations of Anisotropic Particles Accelerated by Neural-Net Predicted Interactions**|B. Rusen Argun et.al.|[2402.12199](http://arxiv.org/abs/2402.12199)|null|
|**2024-02-18**|**AI-assisted inverse design of sequence-ordered high intrinsic thermal conductivity polymers**|Xiang Huang et.al.|[2402.11600](http://arxiv.org/abs/2402.11600)|null|
|**2024-02-18**|**Local and Global Expansivity in Water**|Jan Philipp Gabriel et.al.|[2402.11405](http://arxiv.org/abs/2402.11405)|null|
|**2024-02-16**|**A Gauss-Newton method for iterative optimization of memory kernels for generalized Langevin thermostats in coarse-grained molecular dynamics simulations**|V. Klippenstein et.al.|[2402.10652](http://arxiv.org/abs/2402.10652)|null|
|**2024-02-15**|**Dopant concentration effects on Si $_{1-x}$Ge$_{x}$ crystals for emerging light-source technologies: A molecular dynamics study**|Matthew D. Dickers et.al.|[2402.10120](http://arxiv.org/abs/2402.10120)|null|
|**2024-02-14**|**On the system size dependence of the diffusion coefficients in MD simulations: A simple correction formula for pure dense fluids**|Sergey Khrapak et.al.|[2402.09348](http://arxiv.org/abs/2402.09348)|null|
|**2024-02-14**|**The Photochemistry of Rydberg Excited Cyclobutanone: Photoinduced Processes and Ground State Dynamics**|Julien Eng et.al.|[2402.09140](http://arxiv.org/abs/2402.09140)|null|
|**2024-02-12**|**High Pressure Suppression of Plasticity due to Over-Nucleation of Shear Strain**|Brenden W. Hamilton et.al.|[2402.08104](http://arxiv.org/abs/2402.08104)|null|
|**2024-02-12**|**Nonperturbative Simulation of Anharmonic Rattler Dynamics in Type-I Clathrates with Vibrational Dynamical Mean-Field Theory**|Dipti Jasrasaria et.al.|[2402.08087](http://arxiv.org/abs/2402.08087)|null|
|**2024-02-12**|**Measuring friction from simulations of folded graphene sheets**|Charlie M. Rawlins et.al.|[2402.07768](http://arxiv.org/abs/2402.07768)|null|
|**2024-02-11**|**Cooperating Cracks in Two-Dimensional Crystals**|Shizhe Feng et.al.|[2402.07088](http://arxiv.org/abs/2402.07088)|null|
|**2024-02-10**|**Phase Separation Kinetics and Cluster Dynamics in Two-Dimensional Active Dumbbell Systems**|C. B. Caporusso et.al.|[2402.06998](http://arxiv.org/abs/2402.06998)|null|
|**2024-02-10**|**Mixed equilibrium/nonequilibrium effects govern surface mobility in polymer glasses**|Jianquan Xu et.al.|[2402.06917](http://arxiv.org/abs/2402.06917)|null|
|**2024-02-09**|**Physical, chemical and morphological evolution of incipient soot obtained from molecular dynamics simulation of acetylene pyrolysis**|Khaled Mosharraf Mukut et.al.|[2402.06460](http://arxiv.org/abs/2402.06460)|null|
|**2024-02-09**|**Internal structure of incipient soot from acetylene pyrolysis obtained via molecular dynamics simulations**|Khaled Mosharraf Mukut et.al.|[2402.06456](http://arxiv.org/abs/2402.06456)|null|
|**2024-02-09**|**Radiationless Decay Spectrum of O 1s Double Core Holes in Liquid Water**|Florian Trinter et.al.|[2402.06419](http://arxiv.org/abs/2402.06419)|null|
|**2024-02-09**|**Bayesian Committee Machine Potential for Isothermal-Isobaric Molecular Dynamics Simulations**|Soohaeng Yoo Willow et.al.|[2402.06256](http://arxiv.org/abs/2402.06256)|null|
|**2024-02-09**|**Thermal transport in fullerene-based molecular junctions: Molecular dynamics simulations**|Joanna Li et.al.|[2402.06172](http://arxiv.org/abs/2402.06172)|null|
|**2024-02-08**|**Hydrogen abstraction from metal surfaces: When electron-hole pair excitations strongly affect hot-atom recombination**|Oihana Galparsoro et.al.|[2402.05743](http://arxiv.org/abs/2402.05743)|null|
|**2024-02-07**|**Energy Dissipation to Tungsten Surfaces upon Eley-Rideal Recombination of N2 and H2**|O. Galparsoro et.al.|[2402.04949](http://arxiv.org/abs/2402.04949)|null|
|**2024-02-07**|**Seebeck coefficient of ionic conductors from Bayesian regression analysis**|Enrico Drigo et.al.|[2402.04873](http://arxiv.org/abs/2402.04873)|null|
|**2024-01-30**|**Ionomer structure and component transport in the cathode catalyst layer of PEM fuel cells: A molecular dynamics study**|Yichao Huang et.al.|[2402.04270](http://arxiv.org/abs/2402.04270)|null|
|**2024-02-06**|**On the growing length scale in a replica-coupled glassforming liquid**|Niklas Küchler et.al.|[2402.04205](http://arxiv.org/abs/2402.04205)|null|
|**2024-02-06**|**Electric Field Induced Associations in the Double Layer of Salt-in-Ionic-Liquid Electrolytes**|Daniel M. Markiewitz et.al.|[2402.04039](http://arxiv.org/abs/2402.04039)|null|
|**2024-02-06**|**Scalable Parallel Algorithm for Graph Neural Network Interatomic Potentials in Molecular Dynamics Simulations**|Yutack Park et.al.|[2402.03789](http://arxiv.org/abs/2402.03789)|**[link](https://github.com/mdil-snu/sevennet)**|
|**2024-02-06**|**Enhanced sampling of robust molecular datasets with uncertainty-based collective variables**|Aik Rui Tan et.al.|[2402.03753](http://arxiv.org/abs/2402.03753)|null|
|**2024-02-05**|**Classical molecular dynamics simulations of hydrogen plasmas and development of an analytical statistical model for computational validity assessment**|M. A. Gigosos et.al.|[2402.03401](http://arxiv.org/abs/2402.03401)|null|
|**2024-02-05**|**Elementary vibrational model for thermal conductivity of Lennard-Jones fluids: Applicability domain and accuracy level**|S. A. Khrapak et.al.|[2402.03020](http://arxiv.org/abs/2402.03020)|null|
|**2024-02-04**|**The effect of ligands on the size distribution of copper nanoclusters: insights from molecular dynamics simulations**|Oren Elishav et.al.|[2402.02568](http://arxiv.org/abs/2402.02568)|**[link](https://github.com/oreneli/copper-nucleation)**|
|**2024-02-02**|**Deep learning path-like collective variable for enhanced sampling molecular dynamics**|Thorben Fröhlking et.al.|[2402.01508](http://arxiv.org/abs/2402.01508)|**[link](https://github.com/thorbenf/deeplne)**|
|**2024-02-02**|**Unveiling the early stage evolution of local atomic structures in the crystallization process of a metallic glass**|Lin Ma et.al.|[2402.01325](http://arxiv.org/abs/2402.01325)|null|
|**2024-02-02**|**Conditional Normalizing Flows for Active Learning of Coarse-Grained Molecular Representations**|Henrik Schopmans et.al.|[2402.01195](http://arxiv.org/abs/2402.01195)|null|
|**2024-02-01**|**Anomalous entropy-driven kinetics of dislocation nucleation**|Soumendu Bagchi et.al.|[2402.00810](http://arxiv.org/abs/2402.00810)|null|
|**2024-01-31**|**Perspective: Atomistic Simulations of Water and Aqueous Systems with Machine Learning Potentials**|Amir Omranpour et.al.|[2401.17875](http://arxiv.org/abs/2401.17875)|null|
|**2024-01-31**|**Dimensional Analysis Theory and Molecular Dynamics Simulation of Polypropylene Melt Flow during Injection Molding Process**|Jinrong Zhang et.al.|[2401.17792](http://arxiv.org/abs/2401.17792)|null|
|**2024-01-31**|**Structures and stabilities of mixed clusters of fullerene and coronene molecules**|Naemi Florin et.al.|[2401.17650](http://arxiv.org/abs/2401.17650)|null|
|**2024-01-31**|**Bond breaking and making in mixed clusters of fullerene and coronene molecules following keV-ion impact**|Naemi Florin et.al.|[2401.17638](http://arxiv.org/abs/2401.17638)|null|
|**2024-01-30**|**Development of biochar molecular models with controlled porosity**|Audrey Ngambia et.al.|[2401.17414](http://arxiv.org/abs/2401.17414)|**[link](https://github.com/erastova-group/porous_biochars_models)**|
|**2024-01-29**|**Majority of water hides deep in the interiors of exoplanets**|Haiyang Luo et.al.|[2401.16394](http://arxiv.org/abs/2401.16394)|null|
|**2024-01-29**|**Molecular dynamics simulations of heat transport using machine-learned potentials: A mini review and tutorial on GPUMD with neuroevolution potentials**|Haikuan Dong et.al.|[2401.16249](http://arxiv.org/abs/2401.16249)|**[link](https://gitlab.com/brucefan1983/nep-data)**|
|**2024-02-14**|**Enhancing Protein-Ligand Binding Affinity Predictions using Neural Network Potentials**|Francesc Sabanes Zariquiey et.al.|[2401.16062](http://arxiv.org/abs/2401.16062)|null|
|**2024-02-07**|**Generalized Algorithm for Recognition of Complex Point Defects in Large-Scale β- $\rm {Ga_2O_3}$**|Mengzhi Yan et.al.|[2401.15920](http://arxiv.org/abs/2401.15920)|null|
|**2024-01-26**|**Three phase equilibria of the methane hydrate in NaCl solutions: A simulation study**|S. Blazquez et.al.|[2401.15060](http://arxiv.org/abs/2401.15060)|null|
|**2024-01-26**|**Predictive power of polynomial machine learning potentials for liquid states in 22 elemental systems**|Hayato Wakai et.al.|[2401.14877](http://arxiv.org/abs/2401.14877)|null|
|**2024-01-25**|**Solvation effects on halides core spectra with Multilevel Real-Time quantum embedding**|Jessica A. Martinez B. et.al.|[2401.14548](http://arxiv.org/abs/2401.14548)|null|
|**2024-02-29**|**Threshold displacement energy map of Frenkel pair generation in $\rm Ga_2O_3$ from machine-learning-driven molecular dynamics simulations**|Huan He et.al.|[2401.14039](http://arxiv.org/abs/2401.14039)|null|
|**2024-01-24**|**Experimental and theoretical confirmation of an orthorhombic phase transition in niobium at high pressure and temperature**|Daniel Errandonea et.al.|[2401.13818](http://arxiv.org/abs/2401.13818)|null|
|**2024-02-15**|**Transport Dynamics of Water Molecules Confined between Lipid Membranes**|Minho Lee et.al.|[2401.13253](http://arxiv.org/abs/2401.13253)|null|
|**2024-01-24**|**On the Statistical Mechanics of Mass Accommodation at Liquid-Vapor Interfaces**|Kritanjan Polley et.al.|[2401.13234](http://arxiv.org/abs/2401.13234)|null|
|**2024-01-23**|**Liquid-crystal-like dynamic transition in ferroelectric/dielectric superlattices**|Fernando Gómez-Ortiz et.al.|[2401.13026](http://arxiv.org/abs/2401.13026)|null|
|**2024-01-22**|**Two-dimensional silk**|Chenyang Shi et.al.|[2401.12400](http://arxiv.org/abs/2401.12400)|null|
|**2024-01-21**|**Tunable thermal conductivity of sustainable geopolymers by Si/Al ratio and moisture content: insights from atomistic simulations**|Wenkai Liu et.al.|[2401.11522](http://arxiv.org/abs/2401.11522)|null|
|**2024-01-21**|**Liquid water based optoelectronic computing chip**|Minhui Yang et.al.|[2401.11514](http://arxiv.org/abs/2401.11514)|null|
|**2024-01-20**|**Accurate Description of Ion Migration in Solid-State Ion Conductors from Machine-Learning Molecular Dynamics**|Takeru Miyagawa et.al.|[2401.11244](http://arxiv.org/abs/2401.11244)|null|
|**2024-01-20**|**Salt Effects on Ionic Conductivity Mechanisms in Ethylene Carbonate Electrolytes: Interplay of Viscosity and Ion-ion Relaxations**|Hema Teherpuria et.al.|[2401.11182](http://arxiv.org/abs/2401.11182)|null|
|**2024-01-19**|**Influence of particle size polydispersity on dynamical heterogeneities in dense particle packings**|Rajkumar Biswas et.al.|[2401.10784](http://arxiv.org/abs/2401.10784)|null|
|**2024-01-19**|**Exact analytical algorithm for solvent accessible surface area and derivatives in implicit solvent molecular simulations on GPUs**|Xin Cao et.al.|[2401.10462](http://arxiv.org/abs/2401.10462)|null|
|**2024-01-19**|**Diffusion of intrinsically disordered proteins within viscoelastic membraneless droplets**|Fuga Watanabe et.al.|[2401.10438](http://arxiv.org/abs/2401.10438)|null|
|**2024-02-13**|**Stability of C $_{59}$ Knockout Fragments from Femtoseconds to Infinity**|Michael Gatchell et.al.|[2401.10063](http://arxiv.org/abs/2401.10063)|null|
|**2024-01-19**|**Unified theoretical description of thermal and nonthermal laser-induced ultrafast structural changes in solids**|Bernd Bauerhenne et.al.|[2401.09922](http://arxiv.org/abs/2401.09922)|null|
|**2024-01-18**|**Enhanced thermal conductance at gold-amorphous silicon interfaces**|Julien El Hajj et.al.|[2401.09869](http://arxiv.org/abs/2401.09869)|null|
|**2024-01-18**|**Rational Materials Design for In Operando Electropolymerization of Evolvable Organic Electrochemical Transistors**|J. Y. Gerasimov et.al.|[2401.09835](http://arxiv.org/abs/2401.09835)|null|
|**2024-01-17**|**Entropic force in a dilute solution of real ring polymer chains with different topological structures in a slit of two parallel walls with mixed boundary conditions**|P. Kuterba et.al.|[2401.09353](http://arxiv.org/abs/2401.09353)|null|
|**2024-01-17**|**Role of Elastic Phonon Couplings in Dictating the Thermal Transport across Atomically Sharp SiC/Si Interfaces**|Qinqin He et.al.|[2401.09212](http://arxiv.org/abs/2401.09212)|null|

<p align=right>(<a href=#updated-on-20240320>back to top</a>)</p>

## Transformer

|Publish Date|Title|Authors|PDF|Code|
|---|---|---|---|---|
|**2024-03-18**|**PARMESAN: Parameter-Free Memory Search and Transduction for Dense Prediction Tasks**|Philip Matthias Winter et.al.|[2403.11743](http://arxiv.org/abs/2403.11743)|null|
|**2024-03-18**|**MedMerge: Merging Models for Effective Transfer Learning to Medical Imaging Tasks**|Ibrahim Almakky et.al.|[2403.11646](http://arxiv.org/abs/2403.11646)|null|
|**2024-03-18**|**MISS: Memory-efficient Instance Segmentation Framework By Visual Inductive Priors Flow Propagation**|Chih-Chung Hsu et.al.|[2403.11576](http://arxiv.org/abs/2403.11576)|null|
|**2024-03-18**|**Augment Before Copy-Paste: Data and Memory Efficiency-Oriented Instance Segmentation Framework for Sport-scenes**|Chih-Chung Hsu et.al.|[2403.11572](http://arxiv.org/abs/2403.11572)|null|
|**2024-03-18**|**LOOPer: A Learned Automatic Code Optimizer For Polyhedral Compilers**|Massinissa Merouani et.al.|[2403.11522](http://arxiv.org/abs/2403.11522)|null|
|**2024-03-18**|**Covid-19 detection from CT scans using EfficientNet and Attention mechanism**|Ramy Farag et.al.|[2403.11505](http://arxiv.org/abs/2403.11505)|null|
|**2024-03-18**|**MLVICX: Multi-Level Variance-Covariance Exploration for Chest X-ray Self-Supervised Representation Learning**|Azad Singh et.al.|[2403.11504](http://arxiv.org/abs/2403.11504)|null|
|**2024-03-18**|**SeisFusion: Constrained Diffusion Model with Input Guidance for 3D Seismic Data Interpolation and Reconstruction**|Shuang Wang et.al.|[2403.11482](http://arxiv.org/abs/2403.11482)|**[link](https://github.com/WAL-l/SeisFusion)**|
|**2024-03-18**|**Narrative Feature or Structured Feature? A Study of Large Language Models to Identify Cancer Patients at Risk of Heart Failure**|Ziyi Chen et.al.|[2403.11425](http://arxiv.org/abs/2403.11425)|null|
|**2024-03-18**|**A Deep Learning Method for Beat-Level Risk Analysis and Interpretation of Atrial Fibrillation Patients during Sinus Rhythm**|Jun Lei et.al.|[2403.11405](http://arxiv.org/abs/2403.11405)|null|
|**2024-03-19**|**Automated data processing and feature engineering for deep learning and big data applications: a survey**|Alhassan Mumuni et.al.|[2403.11395](http://arxiv.org/abs/2403.11395)|null|
|**2024-03-17**|**Approximations of the quasi-local Bartnik mass in general relativity**|Leon Escobar-Diaz et.al.|[2403.11372](http://arxiv.org/abs/2403.11372)|null|
|**2024-03-17**|**Solvent-Aware 2D NMR Prediction: Leveraging Multi-Tasking Training and Iterative Self-Training Strategies**|Yunrui Li et.al.|[2403.11353](http://arxiv.org/abs/2403.11353)|null|
|**2024-03-17**|**StainDiffuser: MultiTask Dual Diffusion Model for Virtual Staining**|Tushar Kataria et.al.|[2403.11340](http://arxiv.org/abs/2403.11340)|null|
|**2024-03-17**|**Enhancing Bandwidth Efficiency for Video Motion Transfer Applications using Deep Learning Based Keypoint Prediction**|Xue Bai et.al.|[2403.11337](http://arxiv.org/abs/2403.11337)|null|
|**2024-03-17**|**Diffusion and Multi-Domain Adaptation Methods for Eosinophil Segmentation**|Kevin Lin et.al.|[2403.11323](http://arxiv.org/abs/2403.11323)|null|
|**2024-03-17**|**Advanced Knowledge Extraction of Physical Design Drawings, Translation and conversion to CAD formats using Deep Learning**|Jesher Joshua M et.al.|[2403.11291](http://arxiv.org/abs/2403.11291)|null|
|**2024-03-17**|**Neural-network density functional theory**|Yang Li et.al.|[2403.11287](http://arxiv.org/abs/2403.11287)|null|
|**2024-03-17**|**A Dynamically Weighted Loss Function for Unsupervised Image Segmentation**|Boujemaa Guermazi et.al.|[2403.11266](http://arxiv.org/abs/2403.11266)|null|
|**2024-03-17**|**Simple 2D Convolutional Neural Network-based Approach for COVID-19 Detection**|Chih-Chung Hsu et.al.|[2403.11230](http://arxiv.org/abs/2403.11230)|null|
|**2024-03-17**|**Multiple Teachers-Meticulous Student: A Domain Adaptive Meta-Knowledge Distillation Model for Medical Image Classification**|Shahabedin Nabavi et.al.|[2403.11226](http://arxiv.org/abs/2403.11226)|null|
|**2024-03-17**|**Causality from Bottom to Top: A Survey**|Abraham Itzhak Weinberg et.al.|[2403.11219](http://arxiv.org/abs/2403.11219)|null|
|**2024-03-17**|**CBR - Boosting Adaptive Classification By Retrieval of Encrypted Network Traffic with Out-of-distribution**|Amir Lukach et.al.|[2403.11206](http://arxiv.org/abs/2403.11206)|null|
|**2024-03-17**|**Partitioned Neural Network Training via Synthetic Intermediate Labels**|Cevat Volkan Karadağ et.al.|[2403.11204](http://arxiv.org/abs/2403.11204)|null|
|**2024-03-17**|**Neural Markov Random Field for Stereo Matching**|Tongfan Guan et.al.|[2403.11193](http://arxiv.org/abs/2403.11193)|**[link](https://github.com/aeolusguan/NMRF)**|
|**2024-03-17**|**A Selective Review on Statistical Methods for Massive Data Computation: Distributed Computing, Subsampling, and Minibatch Techniques**|Xuetong Li et.al.|[2403.11163](http://arxiv.org/abs/2403.11163)|null|
|**2024-03-17**|**Spiking Wavelet Transformer**|Yuetong Fang et.al.|[2403.11138](http://arxiv.org/abs/2403.11138)|null|
|**2024-03-17**|**A lightweight deep learning pipeline with DRDA-Net and MobileNet for breast cancer classification**|Mahdie Ahmadi et.al.|[2403.11135](http://arxiv.org/abs/2403.11135)|null|
|**2024-03-17**|**Self-Supervised Quantization-Aware Knowledge Distillation**|Kaiqi Zhao et.al.|[2403.11106](http://arxiv.org/abs/2403.11106)|**[link](https://github.com/kaiqi123/sqakd)**|
|**2024-03-17**|**Bridging Expert Knowledge with Deep Learning Techniques for Just-In-Time Defect Prediction**|Xin Zhou et.al.|[2403.11079](http://arxiv.org/abs/2403.11079)|null|
|**2024-03-17**|**Deep Learning-based Sentiment Analysis in Persian Language**|Mohammad Heydari et.al.|[2403.11069](http://arxiv.org/abs/2403.11069)|null|
|**2024-03-17**|**From Pixels to Predictions: Spectrogram and Vision Transformer for Better Time Series Forecasting**|Zhen Zeng et.al.|[2403.11047](http://arxiv.org/abs/2403.11047)|null|
|**2024-03-16**|**Multiplane Quantitative Phase Imaging Using a Wavelength-Multiplexed Diffractive Optical Processor**|Che-Yung Shen et.al.|[2403.11035](http://arxiv.org/abs/2403.11035)|null|
|**2024-03-16**|**FH-TabNet: Multi-Class Familial Hypercholesterolemia Detection via a Multi-Stage Tabular Deep Learning**|Sadaf Khademi et.al.|[2403.11032](http://arxiv.org/abs/2403.11032)|null|
|**2024-03-16**|**MASSM: An End-to-End Deep Learning Framework for Multi-Anatomy Statistical Shape Modeling Directly From Images**|Janmesh Ukey et.al.|[2403.11008](http://arxiv.org/abs/2403.11008)|null|
|**2024-03-16**|**IoTCO2: Assessing the End-To-End Carbon Footprint of Internet-of-Things-Enabled Deep Learning**|Ahmad Faiz et.al.|[2403.10984](http://arxiv.org/abs/2403.10984)|null|
|**2024-03-16**|**Function-space Parameterization of Neural Networks for Sequential Learning**|Aidan Scannell et.al.|[2403.10929](http://arxiv.org/abs/2403.10929)|**[link](https://github.com/AaltoML/sfr-experiments)**|
|**2024-03-16**|**Automatic location detection based on deep learning**|Anjali Karangiya et.al.|[2403.10912](http://arxiv.org/abs/2403.10912)|null|
|**2024-03-16**|**Could We Generate Cytology Images from Histopathology Images? An Empirical Study**|Soumyajyoti Dey et.al.|[2403.10885](http://arxiv.org/abs/2403.10885)|null|
|**2024-03-16**|**Regularizing CNNs using Confusion Penalty Based Label Smoothing for Histopathology Images**|Somenath Kuiry et.al.|[2403.10881](http://arxiv.org/abs/2403.10881)|null|
|**2024-03-16**|**Neural-Kernel Conditional Mean Embeddings**|Eiki Shimizu et.al.|[2403.10859](http://arxiv.org/abs/2403.10859)|null|
|**2024-03-16**|**RetMIL: Retentive Multiple Instance Learning for Histopathological Whole Slide Image Classification**|Hongbo Chu et.al.|[2403.10858](http://arxiv.org/abs/2403.10858)|null|
|**2024-03-16**|**Deep learning generative model for crystal structure prediction**|Xiaoshan Luo et.al.|[2403.10846](http://arxiv.org/abs/2403.10846)|null|
|**2024-03-16**|**MSI-NeRF: Linking Omni-Depth with View Synthesis through Multi-Sphere Image aided Generalizable Neural Radiance Field**|Dongyu Yan et.al.|[2403.10840](http://arxiv.org/abs/2403.10840)|null|
|**2024-03-16**|**SF(DA) $^2$ : Source-free Domain Adaptation Through the Lens of Data Augmentation**|Uiwon Hwang et.al.|[2403.10834](http://arxiv.org/abs/2403.10834)|**[link](https://github.com/shinyflight/sfda2)**|
|**2024-03-16**|**DUE: Dynamic Uncertainty-Aware Explanation Supervision via 3D Imputation**|Qilong Zhao et.al.|[2403.10831](http://arxiv.org/abs/2403.10831)|null|
|**2024-03-19**|**Securely Fine-tuning Pre-trained Encoders Against Adversarial Examples**|Ziqi Zhou et.al.|[2403.10801](http://arxiv.org/abs/2403.10801)|**[link](https://github.com/cgcl-codes/gen-af)**|
|**2024-03-16**|**CoPlay: Audio-agnostic Cognitive Scaling for Acoustic Sensing**|Yin Li et.al.|[2403.10796](http://arxiv.org/abs/2403.10796)|null|
|**2024-03-16**|**HCF-Net: Hierarchical Context Fusion Network for Infrared Small Object Detection**|Shibiao Xu et.al.|[2403.10778](http://arxiv.org/abs/2403.10778)|**[link](https://github.com/zhengshuchen/hcfnet)**|
|**2024-03-16**|**LIGHTCODE: Light Analytical and Neural Codes for Channels with Feedback**|Sravan Kumar Ankireddy et.al.|[2403.10751](http://arxiv.org/abs/2403.10751)|null|
|**2024-03-15**|**Robust Influence-based Training Methods for Noisy Brain MRI**|Minh-Hao Van et.al.|[2403.10698](http://arxiv.org/abs/2403.10698)|null|
|**2024-03-15**|**EAGLE: An Edge-Aware Gradient Localization Enhanced Loss for CT Image Reconstruction**|Yipeng Sun et.al.|[2403.10695](http://arxiv.org/abs/2403.10695)|**[link](https://github.com/sypsyp97/eagle_loss)**|
|**2024-03-15**|**Hessian-Free Laplace in Bayesian Deep Learning**|James McInerney et.al.|[2403.10671](http://arxiv.org/abs/2403.10671)|null|
|**2024-03-15**|**Leveraging CLIP for Inferring Sensitive Information and Improving Model Fairness**|Miao Zhang et.al.|[2403.10624](http://arxiv.org/abs/2403.10624)|null|
|**2024-03-15**|**Lyapunov Neural Network with Region of Attraction Search**|Zili Wang et.al.|[2403.10621](http://arxiv.org/abs/2403.10621)|null|
|**2024-03-15**|**SurvRNC: Learning Ordered Representations for Survival Prediction using Rank-N-Contrast**|Numan Saeed et.al.|[2403.10603](http://arxiv.org/abs/2403.10603)|null|
|**2024-03-15**|**Medical Unlearnable Examples: Securing Medical Data from Unauthorized Traning via Sparsity-Aware Local Masking**|Weixiang Sun et.al.|[2403.10573](http://arxiv.org/abs/2403.10573)|null|
|**2024-03-14**|**JaxDecompiler: Redefining Gradient-Informed Software Design**|Pierrick Pochelu et.al.|[2403.10571](http://arxiv.org/abs/2403.10571)|null|
|**2024-03-15**|**ATOM: Asynchronous Training of Massive Models for Deep Learning in a Decentralized Environment**|Xiaofeng Wu et.al.|[2403.10504](http://arxiv.org/abs/2403.10504)|null|
|**2024-03-15**|**PnP Restoration with Domain Adaptation for SANS**|Shirin Shoushtari et.al.|[2403.10495](http://arxiv.org/abs/2403.10495)|null|
|**2024-03-15**|**Approximate Nullspace Augmented Finetuning for Robust Vision Transformers**|Haoyang Liu et.al.|[2403.10476](http://arxiv.org/abs/2403.10476)|null|
|**2024-03-15**|**Understanding the Double Descent Phenomenon in Deep Learning**|Marc Lafon et.al.|[2403.10459](http://arxiv.org/abs/2403.10459)|null|
|**2024-03-15**|**Structured Evaluation of Synthetic Tabular Data**|Scott Cheng-Hsin Yang et.al.|[2403.10424](http://arxiv.org/abs/2403.10424)|null|
|**2024-03-15**|**Quantization Avoids Saddle Points in Distributed Optimization**|Yanan Bo et.al.|[2403.10423](http://arxiv.org/abs/2403.10423)|null|
|**2024-03-15**|**A Thorough Comparison of Cross-Encoders and LLMs for Reranking SPLADE**|Hervé Déjean et.al.|[2403.10407](http://arxiv.org/abs/2403.10407)|null|
|**2024-03-15**|**Deep Bi-directional Attention Network for Image Super-Resolution Quality Assessment**|Yixiao Li et.al.|[2403.10406](http://arxiv.org/abs/2403.10406)|null|
|**2024-03-15**|**A comparative study on machine learning approaches for rock mass classification using drilling data**|Tom F. Hansen et.al.|[2403.10404](http://arxiv.org/abs/2403.10404)|null|
|**2024-03-15**|**BirdSet: A Multi-Task Benchmark for Classification in Avian Bioacoustics**|Lukas Rauch et.al.|[2403.10380](http://arxiv.org/abs/2403.10380)|null|
|**2024-03-15**|**Towards a general framework for improving the performance of classifiers using XAI methods**|Andrea Apicella et.al.|[2403.10373](http://arxiv.org/abs/2403.10373)|null|
|**2024-03-15**|**Testing MediaPipe Holistic for Linguistic Analysis of Nonmanual Markers in Sign Languages**|Anna Kuznetsova et.al.|[2403.10367](http://arxiv.org/abs/2403.10367)|null|
|**2024-03-15**|**End-to-end Adaptive Dynamic Subsampling and Reconstruction for Cardiac MRI**|George Yiasemis et.al.|[2403.10346](http://arxiv.org/abs/2403.10346)|null|
|**2024-03-19**|**NetBench: A Large-Scale and Comprehensive Network Traffic Benchmark Dataset for Foundation Models**|Chen Qian et.al.|[2403.10319](http://arxiv.org/abs/2403.10319)|null|
|**2024-03-15**|**Leveraging Neural Radiance Field in Descriptor Synthesis for Keypoints Scene Coordinate Regression**|Huy-Hoang Bui et.al.|[2403.10297](http://arxiv.org/abs/2403.10297)|**[link](https://github.com/ais-lab/descriptorsynthesis4feat2map)**|
|**2024-03-15**|**Deep Learning for Multi-Level Detection and Localization of Myocardial Scars Based on Regional Strain Validated on Virtual Patients**|Müjde Akdeniz et.al.|[2403.10291](http://arxiv.org/abs/2403.10291)|null|
|**2024-03-15**|**Exploring Optical Flow Inclusion into nnU-Net Framework for Surgical Instrument Segmentation**|Marcos Fernández-Rodríguez et.al.|[2403.10216](http://arxiv.org/abs/2403.10216)|null|
|**2024-03-15**|**Learning on JPEG-LDPC Compressed Images: Classifying with Syndromes**|Ahcen Aliouat et.al.|[2403.10202](http://arxiv.org/abs/2403.10202)|null|
|**2024-03-15**|**CoReEcho: Continuous Representation Learning for 2D+time Echocardiography Analysis**|Fadillah Adamsyah Maani et.al.|[2403.10164](http://arxiv.org/abs/2403.10164)|null|
|**2024-03-15**|**Cardiac valve event timing in echocardiography using deep learning and triplane recordings**|Benjamin Strandli Fermann et.al.|[2403.10156](http://arxiv.org/abs/2403.10156)|null|
|**2024-03-15**|**Efficient Multiplayer Battle Game Optimizer for Adversarial Robust Neural Architecture Search**|Rui Zhong et.al.|[2403.10100](http://arxiv.org/abs/2403.10100)|**[link](https://github.com/ruizhong961230/embgo)**|
|**2024-03-15**|**Monkeypox disease recognition model based on improved SE-InceptionV3**|Junzhuo Chen et.al.|[2403.10087](http://arxiv.org/abs/2403.10087)|**[link](https://github.com/jzc777/se-inceptionv3-l2)**|
|**2024-03-15**|**Benchmarking Adversarial Robustness of Image Shadow Removal with Shadow-adaptive Attacks**|Chong Wang et.al.|[2403.10076](http://arxiv.org/abs/2403.10076)|null|
|**2024-03-15**|**PAME: Self-Supervised Masked Autoencoder for No-Reference Point Cloud Quality Assessment**|Ziyu Shan et.al.|[2403.10061](http://arxiv.org/abs/2403.10061)|null|
|**2024-03-15**|**Accurate and Data-Efficient Micro-XRD Phase Identification Using Multi-Task Learning: Application to Hydrothermal Fluids**|Yanfei Li et.al.|[2403.10042](http://arxiv.org/abs/2403.10042)|null|
|**2024-03-15**|**Time-Frequency Jointed Imperceptible Adversarial Attack to Brainprint Recognition with Deep Learning Models**|Hangjie Yi et.al.|[2403.10021](http://arxiv.org/abs/2403.10021)|null|
|**2024-03-15**|**Cardiac Magnetic Resonance 2D+T Short- and Long-axis Segmentation via Spatio-temporal SAM Adaptation**|Zhennong Chen et.al.|[2403.10009](http://arxiv.org/abs/2403.10009)|null|
|**2024-03-15**|**TRG-Net: An Interpretable and Controllable Rain Generator**|Zhiqiang Pang et.al.|[2403.09993](http://arxiv.org/abs/2403.09993)|null|
|**2024-03-15**|**Prediction of Vessel Arrival Time to Pilotage Area Using Multi-Data Fusion and Deep Learning**|Xiaocai Zhang et.al.|[2403.09969](http://arxiv.org/abs/2403.09969)|null|
|**2024-03-15**|**Search-based Ordered Password Generation of Autoregressive Neural Networks**|Min Jin et.al.|[2403.09954](http://arxiv.org/abs/2403.09954)|null|
|**2024-03-15**|**Shifting Focus: From Global Semantics to Local Prominent Features in Swin-Transformer for Knee Osteoarthritis Severity Assessment**|Aymen Sekhri et.al.|[2403.09947](http://arxiv.org/abs/2403.09947)|null|
|**2024-03-14**|**Empowering Healthcare through Privacy-Preserving MRI Analysis**|Al Amin et.al.|[2403.09836](http://arxiv.org/abs/2403.09836)|null|
|**2024-03-14**|**One-shot Learning for MIPs with SOS1 Constraints**|Charly Robinson La Rocca et.al.|[2403.09815](http://arxiv.org/abs/2403.09815)|null|
|**2024-03-14**|**Solving deep-learning density functional theory via variational autoencoders**|Emanuele Costa et.al.|[2403.09788](http://arxiv.org/abs/2403.09788)|null|
|**2024-03-14**|**Phase determination with and without deep learning**|Burak Çivitcioğlu et.al.|[2403.09786](http://arxiv.org/abs/2403.09786)|null|
|**2024-03-14**|**Emotional Intelligence Through Artificial Intelligence : NLP and Deep Learning in the Analysis of Healthcare Texts**|Prashant Kumar Nag et.al.|[2403.09762](http://arxiv.org/abs/2403.09762)|null|
|**2024-03-14**|**SpokeN-100: A Cross-Lingual Benchmarking Dataset for The Classification of Spoken Numbers in Different Languages**|René Groh et.al.|[2403.09753](http://arxiv.org/abs/2403.09753)|**[link](https://github.com/ankilab/spoken-100)**|
|**2024-03-13**|**PICNIQ: Pairwise Comparisons for Natural Image Quality Assessment**|Nicolas Chahine et.al.|[2403.09746](http://arxiv.org/abs/2403.09746)|null|
|**2024-03-13**|**A Sophisticated Framework for the Accurate Detection of Phishing Websites**|Asif Newaz et.al.|[2403.09735](http://arxiv.org/abs/2403.09735)|null|
|**2024-03-14**|**Cloud gap-filling with deep learning for improved grassland monitoring**|Iason Tsardanidis et.al.|[2403.09554](http://arxiv.org/abs/2403.09554)|null|
|**2024-03-14**|**Physics-Informed Neural Network for Volumetric Sound field Reconstruction of Speech Signals**|Marco Olivieri et.al.|[2403.09524](http://arxiv.org/abs/2403.09524)|null|
|**2024-03-14**|**Machine learning activation function parameterization of the occupation numbers for natural orbital functionals based on electron pairing approaches**|L. Franco et.al.|[2403.09463](http://arxiv.org/abs/2403.09463)|null|
|**2024-03-14**|**Deep-learning-assisted optical communication with discretized state space of structural light**|Minyang Zhang et.al.|[2403.09462](http://arxiv.org/abs/2403.09462)|null|
|**2024-03-14**|**Development of control algorithms for mobile robotics focused on their potential use for FPGA-based robots**|Andrés-David Suárez-Gómez et.al.|[2403.09459](http://arxiv.org/abs/2403.09459)|null|
|**2024-03-14**|**Adversarial Fine-tuning of Compressed Neural Networks for Joint Improvement of Robustness and Efficiency**|Hallgrimur Thorsteinsson et.al.|[2403.09441](http://arxiv.org/abs/2403.09441)|**[link](https://github.com/saintslab/adver-fine)**|
|**2024-03-14**|**Efficient Transferability Assessment for Selection of Pre-trained Detectors**|Zhao Wang et.al.|[2403.09432](http://arxiv.org/abs/2403.09432)|null|
|**2024-03-14**|**Low-coercive-field ferroelectric hafnia with mobile domain walls**|Jing Wu et.al.|[2403.09426](http://arxiv.org/abs/2403.09426)|null|
|**2024-03-14**|**Region-based U-net for accelerated training and enhanced precision in deep brain segmentation**|Mengyu Li et.al.|[2403.09414](http://arxiv.org/abs/2403.09414)|null|
|**2024-03-14**|**ConDiSR: Contrastive Disentanglement and Style Regularization for Single Domain Generalization**|Aleksandr Matsun et.al.|[2403.09400](http://arxiv.org/abs/2403.09400)|null|
|**2024-03-14**|**Perspective-Equivariant Imaging: an Unsupervised Framework for Multispectral Pansharpening**|Andrew Wang et.al.|[2403.09327](http://arxiv.org/abs/2403.09327)|null|
|**2024-03-14**|**MOTPose: Multi-object 6D Pose Estimation for Dynamic Video Sequences using Attention-based Temporal Fusion**|Arul Selvam Periyasamy et.al.|[2403.09309](http://arxiv.org/abs/2403.09309)|null|
|**2024-03-14**|**More than words: Advancements and challenges in speech recognition for singing**|Anna Kruspe et.al.|[2403.09298](http://arxiv.org/abs/2403.09298)|null|
|**2024-03-14**|**Identifying Galaxy Cluster Mergers with Deep Neural Networks using Idealized Compton-y and X-ray maps**|Ashleigh R. Arendt et.al.|[2403.09273](http://arxiv.org/abs/2403.09273)|null|
|**2024-03-14**|**Deep Limit Order Book Forecasting**|Antonio Briola et.al.|[2403.09267](http://arxiv.org/abs/2403.09267)|**[link](https://github.com/financialcomputingucl/lobframe)**|
|**2024-03-14**|**A Modified da Vinci Surgical Instrument for OCE based Elasticity Estimation with Deep Learning**|Maximilian Neidhardt et.al.|[2403.09256](http://arxiv.org/abs/2403.09256)|null|
|**2024-03-14**|**Leveraging Constraint Programming in a Deep Learning Approach for Dynamically Solving the Flexible Job-Shop Scheduling Problem**|Imanol Echeverria et.al.|[2403.09249](http://arxiv.org/abs/2403.09249)|null|
|**2024-03-14**|**D-YOLO a robust framework for object detection in adverse weather conditions**|Zihan Chu et.al.|[2403.09233](http://arxiv.org/abs/2403.09233)|null|
|**2024-03-14**|**Generating Feasible and Plausible Counterfactual Explanations for Outcome Prediction of Business Processes**|Alexander Stevens et.al.|[2403.09232](http://arxiv.org/abs/2403.09232)|null|
|**2024-03-14**|**A Robust Semantic Communication System for Image**|Xiang Peng et.al.|[2403.09222](http://arxiv.org/abs/2403.09222)|null|
|**2024-03-14**|**Rectified deep neural networks overcome the curse of dimensionality in the numerical approximation of gradient-dependent semilinear heat equations**|Ariel Neufeld et.al.|[2403.09200](http://arxiv.org/abs/2403.09200)|null|
|**2024-03-14**|**Design of an basis-projected layer for sparse datasets in deep learning training using gc-ms spectra as a case study**|Yu Tang Chang et.al.|[2403.09188](http://arxiv.org/abs/2403.09188)|null|
|**2024-03-14**|**Biophysics Informed Pathological Regularisation for Brain Tumour Segmentation**|Lipei Zhang et.al.|[2403.09136](http://arxiv.org/abs/2403.09136)|null|
|**2024-03-15**|**CardioCaps: Attention-based Capsule Network for Class-Imbalanced Echocardiogram Classification**|Hyunkyung Han et.al.|[2403.09108](http://arxiv.org/abs/2403.09108)|**[link](https://github.com/jihyeonseong/cardiocaps)**|
|**2024-03-14**|**Virtual birefringence imaging and histological staining of amyloid deposits in label-free tissue using autofluorescence microscopy and deep learning**|Xilin Yang et.al.|[2403.09100](http://arxiv.org/abs/2403.09100)|null|
|**2024-03-14**|**An AI-Driven Approach to Wind Turbine Bearing Fault Diagnosis from Acoustic Signals**|Zhao Wang et.al.|[2403.09030](http://arxiv.org/abs/2403.09030)|null|
|**2024-03-14**|**FlexNN: A Dataflow-aware Flexible Deep Learning Accelerator for Energy-Efficient Edge Devices**|Arnab Raha et.al.|[2403.09026](http://arxiv.org/abs/2403.09026)|null|
|**2024-03-13**|**Meta-Learning-Based Fronthaul Compression for Cloud Radio Access Networks**|Ruihua Qiao et.al.|[2403.09004](http://arxiv.org/abs/2403.09004)|**[link](https://github.com/RuihuaQiao/Meta-Learning-Fronthaul-Compression-CRAN)**|
|**2024-03-13**|**7T MRI Synthesization from 3T Acquisitions**|Qiming Cui et.al.|[2403.08979](http://arxiv.org/abs/2403.08979)|**[link](https://github.com/abbasilab/synthetic_7t_mri)**|
|**2024-03-13**|**PathM3: A Multimodal Multi-Task Multiple Instance Learning Framework for Whole Slide Image Classification and Captioning**|Qifeng Zhou et.al.|[2403.08967](http://arxiv.org/abs/2403.08967)|null|
|**2024-03-13**|**Deep Learning Based Dynamics Identification and Linearization of Orbital Problems using Koopman Theory**|George Nehma et.al.|[2403.08965](http://arxiv.org/abs/2403.08965)|null|
|**2024-03-13**|**Using Deep Learning for Morphological Classification in Pigs with a Focus on Sanitary Monitoring**|Eduardo Bedin et.al.|[2403.08962](http://arxiv.org/abs/2403.08962)|null|
|**2024-03-13**|**scVGAE: A Novel Approach using ZINB-Based Variational Graph Autoencoder for Single-Cell RNA-Seq Imputation**|Yoshitaka Inoue et.al.|[2403.08959](http://arxiv.org/abs/2403.08959)|**[link](https://github.com/inoue0426/scVGAE)**|
|**2024-03-15**|**Robust COVID-19 Detection in CT Images with CLIP**|Li Lin et.al.|[2403.08947](http://arxiv.org/abs/2403.08947)|**[link](https://github.com/purdue-m2/covid-19_detection_m2_purdue)**|
|**2024-03-13**|**Federated Data Model**|Xiao Chen et.al.|[2403.08887](http://arxiv.org/abs/2403.08887)|null|
|**2024-03-13**|**Cyclic Data Parallelism for Efficient Parallelism of Deep Neural Networks**|Louis Fournier et.al.|[2403.08837](http://arxiv.org/abs/2403.08837)|null|
|**2024-03-13**|**Structural Positional Encoding for knowledge integration in transformer-based medical process monitoring**|Christopher Irwin et.al.|[2403.08836](http://arxiv.org/abs/2403.08836)|**[link](https://github.com/christopher-irw/proformer_ce)**|
|**2024-03-13**|**Stacking-based deep neural network for player scouting in football 1**|Simon Lacan et.al.|[2403.08835](http://arxiv.org/abs/2403.08835)|null|
|**2024-03-13**|**Spatiotemporal Diffusion Model with Paired Sampling for Accelerated Cardiac Cine MRI**|Shihan Qiu et.al.|[2403.08758](http://arxiv.org/abs/2403.08758)|null|
|**2024-03-13**|**Dynamic computerized tomography using inexact models and motion estimation**|Gesa Sarnighausen et.al.|[2403.08714](http://arxiv.org/abs/2403.08714)|null|
|**2024-03-13**|**Deep Learning for In-Orbit Cloud Segmentation and Classification in Hyperspectral Satellite Data**|Daniel Kovac et.al.|[2403.08695](http://arxiv.org/abs/2403.08695)|null|
|**2024-03-13**|**Self-Supervised Learning for Covariance Estimation**|Tzvi Diskin et.al.|[2403.08662](http://arxiv.org/abs/2403.08662)|null|
|**2024-03-13**|**Leveraging Non-Decimated Wavelet Packet Features and Transformer Models for Time Series Forecasting**|Guy P Nason et.al.|[2403.08630](http://arxiv.org/abs/2403.08630)|null|
|**2024-03-14**|**On the Convergence of Locally Adaptive and Scalable Diffusion-Based Sampling Methods for Deep Bayesian Neural Network Posteriors**|Tim Rensmeyer et.al.|[2403.08609](http://arxiv.org/abs/2403.08609)|null|
|**2024-03-13**|**Patching-based Deep Learning model for the Inpainting of Bragg Coherent Diffraction patterns affected by detectors' gaps**|Matteo Masto et.al.|[2403.08596](http://arxiv.org/abs/2403.08596)|null|
|**2024-03-13**|**Deep Learning based Positioning with Multi-task Learning and Uncertainty-based Fusion**|Anastasios Foliadis et.al.|[2403.08565](http://arxiv.org/abs/2403.08565)|null|
|**2024-03-13**|**Distributed Deep Learning for Modulation Classification in 6G Cell-Free Wireless Networks**|Dieter Verbruggen et.al.|[2403.08563](http://arxiv.org/abs/2403.08563)|null|
|**2024-03-13**|**Causal Graph Neural Networks for Wildfire Danger Prediction**|Shan Zhao et.al.|[2403.08414](http://arxiv.org/abs/2403.08414)|null|
|**2024-03-13**|**FSDR: A Novel Deep Learning-based Feature Selection Algorithm for Pseudo Time-Series Data using Discrete Relaxation**|Mohammad Rahman et.al.|[2403.08403](http://arxiv.org/abs/2403.08403)|null|
|**2024-03-13**|**METER: a mobile vision transformer architecture for monocular depth estimation**|L. Papa et.al.|[2403.08368](http://arxiv.org/abs/2403.08368)|**[link](https://github.com/lorenzopapa5/meter)**|
|**2024-03-13**|**From human experts to machines: An LLM supported approach to ontology and knowledge graph construction**|Vamsi Krishna Kommineni et.al.|[2403.08345](http://arxiv.org/abs/2403.08345)|null|
|**2024-03-13**|**Physics-Informed Deep Learning for Motion-Corrected Reconstruction of Quantitative Brain MRI**|Hannah Eichhorn et.al.|[2403.08298](http://arxiv.org/abs/2403.08298)|**[link](https://github.com/hannaheichhorn/phimo)**|
|**2024-03-13**|**Performance assessment of the effective core potentials under the Fermionic neural network: first and second row elements**|Mengsa Wang et.al.|[2403.08287](http://arxiv.org/abs/2403.08287)|null|
|**2024-03-13**|**Optimized Detection and Classification on GTRSB: Advancing Traffic Sign Recognition with Convolutional Neural Networks**|Dhruv Toshniwal et.al.|[2403.08283](http://arxiv.org/abs/2403.08283)|null|
|**2024-03-13**|**VIGFace: Virtual Identity Generation Model for Face Image Synthesis**|Minsoo Kim et.al.|[2403.08277](http://arxiv.org/abs/2403.08277)|null|
|**2024-03-13**|**Sketch2Manga: Shaded Manga Screening from Sketch with Diffusion Models**|Jian Lin et.al.|[2403.08266](http://arxiv.org/abs/2403.08266)|null|
|**2024-03-13**|**Research on the Application of Deep Learning-based BERT Model in Sentiment Analysis**|Yichao Wu et.al.|[2403.08217](http://arxiv.org/abs/2403.08217)|null|
|**2024-03-13**|**P2LHAP:Wearable sensor-based human activity recognition, segmentation and forecast through Patch-to-Label Seq2Seq Transformer**|Shuangjian Li et.al.|[2403.08214](http://arxiv.org/abs/2403.08214)|null|
|**2024-03-13**|**Advancing Security in AI Systems: A Novel Approach to Detecting Backdoors in Deep Neural Networks**|Khondoker Murad Hossain et.al.|[2403.08208](http://arxiv.org/abs/2403.08208)|null|
|**2024-03-13**|**SpeechColab Leaderboard: An Open-Source Platform for Automatic Speech Recognition Evaluation**|Jiayu Du et.al.|[2403.08196](http://arxiv.org/abs/2403.08196)|**[link](https://github.com/speechcolab/leaderboard)**|
|**2024-03-13**|**EM-TTS: Efficiently Trained Low-Resource Mongolian Lightweight Text-to-Speech**|Ziqi Liang et.al.|[2403.08164](http://arxiv.org/abs/2403.08164)|null|

<p align=right>(<a href=#updated-on-20240320>back to top</a>)</p>

## bind

|Publish Date|Title|Authors|PDF|Code|
|---|---|---|---|---|
|**2006-08-17**|**The role of binding entropy in the refinement of protein-ligand docking predictions: analysis based on the use of 11 scoring functions**|Anatoly M. Ruvinsky et.al.|[physics/0608186](http://arxiv.org/abs/physics/0608186)|null|
|**2004-09-09**|**Method for Computing Protein Binding Affinity**|Charles F. F. Karney et.al.|[cond-mat/0401348](http://arxiv.org/abs/cond-mat/0401348)|null|
|**2024-02-21**|**Re-Dock: Towards Flexible and Realistic Molecular Docking with Diffusion Bridge**|Yufei Huang et.al.|[2402.11459](http://arxiv.org/abs/2402.11459)|null|
|**2024-02-14**|**Enhancing Protein-Ligand Binding Affinity Predictions using Neural Network Potentials**|Francesc Sabanes Zariquiey et.al.|[2401.16062](http://arxiv.org/abs/2401.16062)|null|
|**2024-02-01**|**A Multi-Grained Symmetric Differential Equation Model for Learning Protein-Ligand Binding Dynamics**|Shengchao Liu et.al.|[2401.15122](http://arxiv.org/abs/2401.15122)|null|
|**2023-11-09**|**Protein-ligand binding representation learning from fine-grained interactions**|Shikun Feng et.al.|[2311.16160](http://arxiv.org/abs/2311.16160)|null|
|**2023-12-19**|**DiffBindFR: An SE(3) Equivariant Network for Flexible Protein-Ligand Docking**|Jintao Zhu et.al.|[2311.15201](http://arxiv.org/abs/2311.15201)|**[link](https://github.com/HBioquant/DiffBindFR)**|
|**2023-09-22**|**HydraScreen: A Generalizable Structure-Based Deep Learning Approach to Drug Discovery**|Alvaro Prat et.al.|[2311.12814](http://arxiv.org/abs/2311.12814)|null|
|**2023-11-19**|**A Universal Framework for Accurate and Efficient Geometric Deep Learning of Molecular Systems**|Shuo Zhang et.al.|[2311.11228](http://arxiv.org/abs/2311.11228)|**[link](https://github.com/XieResearchGroup/Physics-aware-Multiplex-GNN)**|
|**2024-01-09**|**FABind: Fast and Accurate Protein-Ligand Binding**|Qizhi Pei et.al.|[2310.06763](http://arxiv.org/abs/2310.06763)|**[link](https://github.com/qizhipei/fabind)**|
|**2023-11-04**|**Harmonic Self-Conditioned Flow Matching for Multi-Ligand Docking and Binding Site Design**|Hannes Stärk et.al.|[2310.05764](http://arxiv.org/abs/2310.05764)|**[link](https://github.com/hannesstark/flowsite)**|
|**2023-12-18**|**PharmacoNet: Accelerating Large-Scale Virtual Screening by Deep Pharmacophore Modeling**|Seonghwan Seo et.al.|[2310.00681](http://arxiv.org/abs/2310.00681)|**[link](https://github.com/seonghwanseo/molvoxel)**|
|**2024-02-26**|**A hybrid quantum-classical fusion neural network to improve protein-ligand binding affinity predictions for drug discovery**|L. Domingo et.al.|[2309.03919](http://arxiv.org/abs/2309.03919)|null|
|**2023-08-18**|**Leak Proof PDBBind: A Reorganized Dataset of Protein-Ligand Complexes for More Generalizable Binding Affinity Prediction**|Jie Li et.al.|[2308.09639](http://arxiv.org/abs/2308.09639)|null|
|**2023-06-28**|**Protein-DNA binding sites prediction based on pre-trained protein language model and contrastive learning**|Yufan Liu et.al.|[2306.15912](http://arxiv.org/abs/2306.15912)|**[link](https://github.com/yandrewl/clape)**|
|**2023-12-20**|**Multi-task Bioassay Pre-training for Protein-ligand Binding Affinity Prediction**|Jiaxian Yan et.al.|[2306.04886](http://arxiv.org/abs/2306.04886)|**[link](https://github.com/jiaxianyan/mbp)**|
|**2023-03-20**|**Validation of the Alchemical Transfer Method for the Estimation of Relative Binding Affinities of Molecular Series**|Francesc Sabanés Zariquiey et.al.|[2303.11065](http://arxiv.org/abs/2303.11065)|**[link](https://github.com/compsciencelab/atm_benchmark)**|
|**2023-12-12**|**Unsupervised Protein-Ligand Binding Energy Prediction via Neural Euler's Rotation Equation**|Wengong Jin et.al.|[2301.10814](http://arxiv.org/abs/2301.10814)|**[link](https://github.com/wengong-jin/dsmbind)**|
|**2023-01-15**|**Geometric Graph Learning with Extended Atom-Types Features for Protein-Ligand Binding Affinity Prediction**|Md Masud Rana et.al.|[2301.06194](http://arxiv.org/abs/2301.06194)|**[link](https://github.com/nguyenlabuky/ggl-eta-score)**|
|**2023-08-02**|**Protein-Ligand Complex Generator & Drug Screening via Tiered Tensor Transform**|Jonathan P. Mailoa et.al.|[2301.00984](http://arxiv.org/abs/2301.00984)|**[link](https://github.com/tencent-quantum-lab/3t)**|
|**2022-12-27**|**SVSBI: Sequence-based virtual screening of biomolecular interactions**|Li Shen et.al.|[2212.13617](http://arxiv.org/abs/2212.13617)|**[link](https://github.com/weilabmsu/svs)**|
|**2023-03-28**|**HAC-Net: A Hybrid Attention-Based Convolutional Neural Network for Highly Accurate Protein-Ligand Binding Affinity Prediction**|Gregory W. Kyro et.al.|[2212.12440](http://arxiv.org/abs/2212.12440)|**[link](https://github.com/gregory-kyro/hac-net)**|
|**2023-04-26**|**Geometry-Complete Perceptron Networks for 3D Molecular Graphs**|Alex Morehead et.al.|[2211.02504](http://arxiv.org/abs/2211.02504)|**[link](https://github.com/bioinfomachinelearning/gcpnet)**|
|**2022-10-27**|**Predicting Protein-Ligand Binding Affinity with Equivariant Line Graph Network**|Yiqiang Yi et.al.|[2210.16098](http://arxiv.org/abs/2210.16098)|null|
|**2023-09-25**|**Exploring Data-Driven Chemical SMILES Tokenization Approaches to Identify Key Protein-Ligand Binding Moieties**|Asu Büşra Temizer et.al.|[2210.14642](http://arxiv.org/abs/2210.14642)|null|
|**2023-06-12**|**Holographic-(V)AE: an end-to-end SO(3)-Equivariant (Variational) Autoencoder in Fourier Space**|Gian Marco Visani et.al.|[2209.15567](http://arxiv.org/abs/2209.15567)|**[link](https://github.com/gvisani/holographic-vae)**|
|**2022-09-18**|**Predicting Protein-Ligand Binding Affinity via Joint Global-Local Interaction Modeling**|Yang Zhang et.al.|[2209.13014](http://arxiv.org/abs/2209.13014)|null|
|**2023-06-03**|**From Static to Dynamic Structures: Improving Binding Affinity Prediction with a Graph-Based Deep Learning Model**|Yaosen Min et.al.|[2208.10230](http://arxiv.org/abs/2208.10230)|**[link](https://github.com/minys233/dynafomer)**|
|**2022-06-27**|**A fully differentiable ligand pose optimization framework guided by deep learning and traditional scoring functions**|Zechen Wang et.al.|[2206.13345](http://arxiv.org/abs/2206.13345)|**[link](https://github.com/zchwang/deeprmsd-vina_optimization)**|
|**2022-08-24**|**SHREC 2022: Protein-ligand binding site recognition**|Luca Gagliardi et.al.|[2206.06035](http://arxiv.org/abs/2206.06035)|**[link](https://github.com/concept-lab/shrec22_proteinligandbenchmark)**|
|**2023-11-19**|**Efficient and Accurate Physics-aware Multiplex Graph Neural Networks for 3D Small Molecules and Macromolecule Complexes**|Shuo Zhang et.al.|[2206.02789](http://arxiv.org/abs/2206.02789)|**[link](https://github.com/XieResearchGroup/Physics-aware-Multiplex-GNN)**|
|**2022-12-09**|**Adding alchemical variables to metadynamics to enhance sampling in free energy calculations**|Wei-Tse Hsu et.al.|[2206.01329](http://arxiv.org/abs/2206.01329)|null|
|**2022-06-01**|**EISA-Score: Element Interactive Surface Area Score for Protein-Ligand Binding Affinity Prediction**|Md Masud Rana et.al.|[2206.00611](http://arxiv.org/abs/2206.00611)|**[link](https://github.com/nguyenlabuky/eisa-score)**|
|**2022-05-31**|**Contrastive Representation Learning for 3D Protein Structures**|Pedro Hermosilla et.al.|[2205.15675](http://arxiv.org/abs/2205.15675)|null|
|**2023-01-02**|**A New Formulation for the Concerted Alchemical Calculation of van der Waals and Coulomb Components of Solvation Free Energies**|Gabriela B. Correa et.al.|[2205.11464](http://arxiv.org/abs/2205.11464)|**[link](https://github.com/atoms-ufrj/jctc-lbf-paper)**|
|**2022-03-25**|**Meaningful machine learning models and machine-learned pharmacophores from fragment screening campaigns**|Carl Poelking et.al.|[2204.06348](http://arxiv.org/abs/2204.06348)|null|
|**2022-05-13**|**GROMACS in the cloud: A global supercomputer to speed up alchemical drug design**|Carsten Kutzner et.al.|[2201.06372](http://arxiv.org/abs/2201.06372)|null|
|**2022-11-09**|**AI-Bind: Improving Binding Predictions for Novel Protein Targets and Ligands**|Ayan Chatterjee et.al.|[2112.13168](http://arxiv.org/abs/2112.13168)|**[link](https://github.com/Barabasi-Lab/AI-Bind)**|
|**2022-01-26**|**Generating 3D Molecules Conditional on Receptor Binding Sites with Deep Generative Models**|Matthew Ragoza et.al.|[2110.15200](http://arxiv.org/abs/2110.15200)|**[link](https://github.com/mattragoza/liGAN)**|
|**2021-10-15**|**Quantum Computational Quantification of Protein-Ligand Interactions**|Josh John Mellor Kirsopp et.al.|[2110.08163](http://arxiv.org/abs/2110.08163)|null|
|**2021-09-03**|**Ligand-induced protein dynamics differences correlate with protein-ligand binding affinities: An unsupervised deep learning approach**|Ikki Yasuda et.al.|[2109.01339](http://arxiv.org/abs/2109.01339)|null|
|**2021-08-10**|**On-chip Acousto Thermal Shift Assay for Rapid and Sensitive Assessment of Protein Thermodynamic Stability**|Yonghui Ding et.al.|[2108.04935](http://arxiv.org/abs/2108.04935)|null|
|**2021-08-26**|**The Organization of Chaos into a Molecular Trap that Supervises Ligand-Interaction, Selection and Steric Guidance Similar to Events in Black Holes**|Leroy K. Davis et.al.|[2108.01223](http://arxiv.org/abs/2108.01223)|null|
|**2021-07-21**|**Structure-aware Interactive Graph Neural Networks for the Prediction of Protein-Ligand Binding Affinity**|Shuangli Li et.al.|[2107.10670](http://arxiv.org/abs/2107.10670)|**[link](https://github.com/agave233/SIGN)**|
|**2021-07-09**|**A Point Cloud-Based Deep Learning Strategy for Protein-Ligand Binding Affinity Prediction**|Yeji Wang et.al.|[2107.04340](http://arxiv.org/abs/2107.04340)|null|
|**2021-11-12**|**Best practices for constructing, preparing, and evaluating protein-ligand binding affinity benchmarks**|David F. Hahn et.al.|[2105.06222](http://arxiv.org/abs/2105.06222)|**[link](https://github.com/openforcefield/FE-Benchmarks-Best-Practices)**|
|**2021-06-01**|**High-Throughput Virtual Screening of Small Molecule Inhibitors for SARS-CoV-2 Protein Targets with Deep Fusion Models**|Garrett A. Stevenson et.al.|[2104.04547](http://arxiv.org/abs/2104.04547)|null|
|**2021-03-22**|**OnionNet-2: A Convolutional Neural Network Model for Predicting Protein-Ligand Binding Affinity based on Residue-Atom Contacting Shells**|Zechen Wang et.al.|[2103.11664](http://arxiv.org/abs/2103.11664)|**[link](https://github.com/zchwang/OnionNet-2)**|
|**2020-11-20**|**Ollivier persistent Ricci curvature (OPRC) based molecular representation for drug design**|JunJie Wee et.al.|[2011.10281](http://arxiv.org/abs/2011.10281)|**[link](https://github.com/ExpectozJJ/Persistent-Ollivier-Ricci-Curvature)**|
|**2020-09-30**|**Kinetics and Free Energy of Ligand Dissociation Using Weighted Ensemble Milestoning**|Dhiman Ray et.al.|[2007.09325](http://arxiv.org/abs/2007.09325)|**[link](https://github.com/dhimanray/WEM-Ligand)**|
|**2020-07-08**|**Attribution Methods Reveal Flaws in Fingerprint-Based Virtual Screening**|Vikram Sundar et.al.|[2007.01436](http://arxiv.org/abs/2007.01436)|null|
|**2020-05-12**|**InteractionNet: Modeling and Explaining of Noncovalent Protein-Ligand Interactions with Noncovalent Graph Neural Network and Layer-Wise Relevance Propagation**|Hyeoncheol Cho et.al.|[2005.13438](http://arxiv.org/abs/2005.13438)|null|
|**2020-05-17**|**Improved Protein-ligand Binding Affinity Prediction with Structure-Based Deep Fusion Inference**|Derek Jones et.al.|[2005.07704](http://arxiv.org/abs/2005.07704)|**[link](https://github.com/LLNL/FAST)**|
|**2020-07-27**|**Site2Vec: a reference frame invariant algorithm for vector embedding of protein-ligand binding sites**|Arnab Bhadra et.al.|[2003.08149](http://arxiv.org/abs/2003.08149)|null|
|**2023-01-13**|**The stability of persistent homology of hypergraphs**|Shiquan Ren et.al.|[2002.02237](http://arxiv.org/abs/2002.02237)|null|
|**2020-02-03**|**Persistent spectral based machine learning (PerSpect ML) for drug design**|Zhenyu Meng et.al.|[2002.00582](http://arxiv.org/abs/2002.00582)|null|
|**2020-01-09**|**Split Optimization for Protein/Ligand Binding Models**|Brian Davis et.al.|[2001.03207](http://arxiv.org/abs/2001.03207)|null|
|**2019-11-22**|**Schrödinger-ANI: An Eight-Element Neural Network Interaction Potential with Greatly Expanded Coverage of Druglike Chemical Space**|James M. Stevenson et.al.|[1912.05079](http://arxiv.org/abs/1912.05079)|null|
|**2019-12-03**|**A review of mathematical representations of biomolecules**|Duc D Nguyen et.al.|[1912.04724](http://arxiv.org/abs/1912.04724)|null|
|**2019-12-01**|**DeepAtom: A Framework for Protein-Ligand Binding Affinity Prediction**|Yanjun Li et.al.|[1912.00318](http://arxiv.org/abs/1912.00318)|**[link](https://github.com/YanjunLi-CS/DeepAtom_SupplementaryMaterials)**|
|**2019-11-03**|**Are 2D fingerprints still valuable for drug discovery?**|Kaifu Gao et.al.|[1911.00930](http://arxiv.org/abs/1911.00930)|null|
|**2019-10-07**|**Combining docking pose rank and structure with deep learning improves protein-ligand binding mode prediction**|Joseph A. Morrone et.al.|[1910.02845](http://arxiv.org/abs/1910.02845)|null|
|**2020-02-04**|**Interactive molecular dynamics in virtual reality for accurate flexible protein-ligand docking**|Helen M. Deeks et.al.|[1908.07395](http://arxiv.org/abs/1908.07395)|null|
|**2019-06-06**|**OnionNet: a multiple-layer inter-molecular contact based convolutional neural network for protein-ligand binding affinity prediction**|Liangzhen Zheng et.al.|[1906.02418](http://arxiv.org/abs/1906.02418)|**[link](https://github.com/zhenglz/onionnet)**|
|**2020-03-28**|**Improving detection of protein-ligand binding sites with 3D segmentation**|Marta M. Stepniewska-Dziubinska et.al.|[1904.06517](http://arxiv.org/abs/1904.06517)|**[link](https://gitlab.com/cheminfIBB/kalasanty)**|
|**2019-05-01**|**Interactive molecular dynamics in virtual reality from quantum chemistry to drug binding: An open-source multi-person framework**|Michael O'Connor et.al.|[1902.01827](http://arxiv.org/abs/1902.01827)|null|
|**2018-12-20**|**Algebraic graph learning of protein-ligand binding affinity**|Duc Duy Nguyen et.al.|[1812.08328](http://arxiv.org/abs/1812.08328)|null|
|**2019-05-19**|**Using Attribution to Decode Dataset Bias in Neural Network Models for Chemistry**|Kevin McCloskey et.al.|[1811.11310](http://arxiv.org/abs/1811.11310)|null|
|**2020-12-18**|**ChemBoost: A chemical language based approach for protein-ligand binding affinity prediction**|Rıza Özçelik et.al.|[1811.00761](http://arxiv.org/abs/1811.00761)|**[link](https://github.com/boun-tabi/chemboost)**|
|**2018-07-29**|**Persistent cohomology for data with multicomponent heterogeneous information**|Zixuan Cang et.al.|[1807.11120](http://arxiv.org/abs/1807.11120)|null|
|**2018-06-11**|**DG-GL: Differential geometry based geometric learning of molecular datasets**|Duc Duy Nguyen et.al.|[1806.03765](http://arxiv.org/abs/1806.03765)|null|
|**2018-10-22**|**PotentialNet for Molecular Property Prediction**|Evan N. Feinberg et.al.|[1803.04465](http://arxiv.org/abs/1803.04465)|null|
|**2018-01-03**|**Development and evaluation of a deep learning model for protein-ligand binding affinity prediction**|Marta M. Stepniewska-Dziubinska et.al.|[1712.07042](http://arxiv.org/abs/1712.07042)|**[link](https://gitlab.com/cheminfIBB/pafnucy)**|
|**2017-12-01**|**Shape Complementarity Analysis for Objects of Arbitrary Shape**|Morad Behandish et.al.|[1712.00238](http://arxiv.org/abs/1712.00238)|null|
|**2017-11-02**|**Dimensionality reduction methods for molecular simulations**|Stefan Doerr et.al.|[1710.10629](http://arxiv.org/abs/1710.10629)|null|
|**2017-08-27**|**Representability of algebraic topology for biomolecules in machine learning based scoring and virtual screening**|Zixuan Cang et.al.|[1708.08135](http://arxiv.org/abs/1708.08135)|null|
|**2017-03-31**|**Rigidity strengthening is a vital mechanism for protein-ligand binding**|Duc Duy Nguyen et.al.|[1704.05883](http://arxiv.org/abs/1704.05883)|null|
|**2017-03-31**|**TopologyNet: Topology based deep convolutional neural networks for biomolecular property predictions**|Zixuan Cang et.al.|[1704.00063](http://arxiv.org/abs/1704.00063)|null|
|**2017-03-31**|**Topological fingerprints reveal protein-ligand binding mechanism**|Zixuan Cang et.al.|[1703.10982](http://arxiv.org/abs/1703.10982)|null|
|**2017-03-31**|**Feature functional theory - binding predictor (FFT-BP) for the blind prediction of binding free energies**|Bao Wang et.al.|[1703.10927](http://arxiv.org/abs/1703.10927)|null|
|**2017-03-30**|**Atomic Convolutional Networks for Predicting Protein-Ligand Binding Affinity**|Joseph Gomes et.al.|[1703.10603](http://arxiv.org/abs/1703.10603)|null|
|**2017-01-18**|**Free Energy Computation by Monte Carlo Integration**|Matthew Clark et.al.|[1701.05073](http://arxiv.org/abs/1701.05073)|null|
|**2016-12-08**|**Protein-Ligand Scoring with Convolutional Neural Networks**|Matthew Ragoza et.al.|[1612.02751](http://arxiv.org/abs/1612.02751)|**[link](https://github.com/gnina/gnina)**|
|**2015-01-17**|**Numerical calculation of protein-ligand binding rates through solution of the Smoluchowski equation using smooth particle hydrodynamics**|Wenxiao Pan et.al.|[1501.04240](http://arxiv.org/abs/1501.04240)|null|
|**2013-07-03**|**How hydrophobic drying forces impact the kinetics of molecular recognition**|Jagannath Mondal et.al.|[1305.7505](http://arxiv.org/abs/1305.7505)|null|
|**2008-11-24**|**Absolute FKBP binding affinities obtained via non-equilibrium unbinding simulations**|F. Marty Ytreberg et.al.|[0710.5132](http://arxiv.org/abs/0710.5132)|null|

<p align=right>(<a href=#updated-on-20240320>back to top</a>)</p>

[contributors-shield]: https://img.shields.io/github/contributors/Vincentqyw/cv-arxiv-daily.svg?style=for-the-badge
[contributors-url]: https://github.com/Vincentqyw/cv-arxiv-daily/graphs/contributors
[forks-shield]: https://img.shields.io/github/forks/Vincentqyw/cv-arxiv-daily.svg?style=for-the-badge
[forks-url]: https://github.com/Vincentqyw/cv-arxiv-daily/network/members
[stars-shield]: https://img.shields.io/github/stars/Vincentqyw/cv-arxiv-daily.svg?style=for-the-badge
[stars-url]: https://github.com/Vincentqyw/cv-arxiv-daily/stargazers
[issues-shield]: https://img.shields.io/github/issues/Vincentqyw/cv-arxiv-daily.svg?style=for-the-badge
[issues-url]: https://github.com/Vincentqyw/cv-arxiv-daily/issues

